---
title: "Descriptive epidemiology of risk factors and enteric non-typhoidal salmonella in children under 5 years in Chikwawa"
format:    
  pdf:
    geometry:
      - top=20mm
      - bottom=20mm
      - left=20mm
      - right=20mm
    toc: true
    lof: true
    lot: true
    number-sections: true
    colorlinks: true
    include-in-header:
      text: |
        \usepackage{typearea}
    cite-method: citeproc
    bibliography: references.bib
    csl: diabetologia.csl
editor: visual
---

## Introduction

As illustrated in the previous chapter, understanding the relationship and interactions between non-typhoidal salmonella infections and host susceptibility factors such as malaria, HIV, severe acute malnutrition, and anaemia, is key to informing the focus of public health interventions. Previous work has been done to understand the underlying immune mechanisms leading to susceptibility by factors such as malaria [@nyirenda_immunological_2018, @nyirenda_loss_2017, @cunnington_suppression_2010, @gondwe_importance_2010], haemolysis [@cunnington2011], sickle cell disease [@hand_wl_deficiency_1977, @sanhadji1988], HIV [@gordon2002, @okoro2012, @gordon_invasive_2010, @maclennan_dysregulated_2010] and severe acute malnutrition (SAM) [@vásquez-garibay2002], as described in chapter 1. Asymptomatic carriage of NTS subclades associated with invasive disease have been demonstrated in 0.79% of stools from health children recruited as controls from a population living in Mukuru informal settlements in Kenya [@kariuki_high_2020]. NTS was found in the stools of 8-9% of children 6 - 18 months, 48% of which were *salmonella* typhimurium ST313, during monthly sampling in a longitudinal community cohort of 50 Malawian children [@nyirenda2015] as described in chapter 1. However, no large scale community stool testing for NTS in children under 5 years has been carried out in endemic settings, with simultaneous collection of data on host susceptibility factors such as malaria, malnutrition, anaemia and sickle cell disease, combined with markers of immune response to NTS. Previous seroepidemiological studies have relied on data collected at a single time point, and may miss crucial interactions of seasonal changes on exposure to enteric pathogens, seasonality of host susceptibility factors, and seasonality of immune responses to NTS enteric exposures.

This chapter sets out the descriptive epidemiological findings from the SAiNTS Malawi study, describing the spatial and temporal distribution of host susceptibility factors to NTS in children 0 - 5 years recruited in the Chikwawa district, Southern Malawi.

## Methods

As described in chapter 2, children were recruited prospectively randomly sampled from census data collected by the EPIMAL study across two clusters; St Montfort that was randomly allocated to receive the malaria RTS,S/AS01~E~ vaccine and Chikwawa that was randomly allocated to not receive the RTS,S/AS01~E~ vaccine. Two stage sampling was carried out using 2-stage sampling technique; Stage 1: sampling enumeration areas stratified by baseline malaria prevalence from the malaria indicator survey data for Chikwawa (@fig-MIS2017) [@malaria2017], and based on probability proportional to population size of the EA (PPS), and Stage 2: households containing children 0–5 years were randomly sampled from selected Enumeration Area (EA). Any child eligible in the household was invited to participate in the study. Children were recruited and samples were collected for stool, serology and host susceptibility data (malaria, anaemia, SAM, sickle cell) at day 1 and repeat serology and host susceptibility data at day 90. Serum samples were tested for lipopolysaccharide O-antigen immunoglobin-G antibody to the main serovars causing invasive NTS disease (*Salmonella* Typhimurium and Enteritidis). Host susceptibility factors were measured through anthropometric measurements, malaria (rapid diagnostic tests \[RDT\] and thick blood films), haemocue and sickle RDT. Stool samples were processed for Salmonella culture and pan-Salmonella PCR (TTR-primer). Data on medical history, vaccination history, socioeconomic status, water and sanitation and GPS location of households were collected by electronic case report forms on tablets at the household and blood sampling tent.

### Statistical methods

A proxy poverty score was derived as used in the SCALE trial [@feasey2023] to assess poverty in research studies in Malawi, with different variables selected to be appropriate for urban verses rural settings. Logistic regression was used to select the 10 most important variables from the Malawi 4^th^ Integrated Household Survey 2016-2017 (<https://microdata.worldbank.org/index.php/catalog/2936>) associated with poverty. Sequential models were run from 10 down to 6 variables to try and use the minimum variables possible to create a proxy poverty score. The model was run with and without the two ennumerator-defined variables (housing structure and floor composition) due to these variables being observed by fieldworkers and might not be able to be asked of participants. Bootstrapping was used to validate the model. The new model was compared with proxy poverty score variables from 1998 HIS, and the new variables improved the model fit. The coefficients from the final 10 variables were used as weights to form the final proxy poverty score.

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 6, fig.height=7, out.width = "6.5in"}

# setup
library(rio)
library(here)
library(tidyverse)
library(janitor)
library(Hmisc)
library(flextable)
library(gt)
library(gtsummary)

master <- import(here("data", "5_data", "master.csv")) %>% 
  mutate(ta_code = substr(ta_code,5,8)) 
         # cluster = ifelse(cluster==1,"Chikwawa","St Montfort"))
seqdat <- import(here("data", "5_data", "2023.06.29.SAiNTS_seq_results.xlsx"))
watsan <- import(here("data", "5_data", "WatSan.xlsx"))

master2 <- master %>% left_join(watsan)
table(master2$drink_wat_desc)

watsancheck <- master2 %>% 
  select(c(pid, drinking_water_src...7, drink_wat_desc, cooking_water_src, cooking_wat_desc, toilet_type...3, toilet_type...4, share_with, share_toilet_des, bathing_water_src, bathing_wat_desc))

EA_clusEA <- master %>% 
  count(cluster, ta_code) 
EA_clus <- EA_clusEA %>% 
  count(cluster) 
n <- count(master)
nclus <- master %>% count(cluster)
nchik <- nclus %>% filter(cluster=="Chikwawa")
nmont <- nclus %>% filter(cluster=="St Montfort")

NclusEA <- EA_clusEA %>% 
  group_by(cluster) %>% 
  summarise(mean_no = mean(n, na.rm=T),
            range = range(n)) %>% 
  mutate(range_type = rep("low", "high",2)) %>% 
  pivot_wider(names_from = "range_type", values_from = "range") %>% 
  mutate(range = substr(low,2,10)) %>% 
  ungroup()

EAperClus <- EA_clus %>%  left_join(NclusEA)
chiN <- EAperClus %>% filter(cluster=="Chikwawa")
StMontN <- EAperClus %>% filter(cluster=="St Montfort")

ageclus <- master %>% 
  group_by(cluster) %>% 
  summarise(age_mean = mean(calc_age_months, na.rm=T),
            age_sd = sd(calc_age_months, na.rm=T))
ageChi <- ageclus %>% filter(cluster=="Chikwawa")
ageMon <- ageclus %>% filter(cluster=="St Montfort")

pov <- master %>% 
  group_by(cluster) %>% 
  summarise(mean = mean(pov_score, na.rm=T),
            sd = sd(pov_score, na.rm=T))
povChi <- pov %>% filter(cluster=="Chikwawa")
povMon <- pov %>% filter(cluster=="St Montfort")

water <- master %>% select(c(cluster, toilet_type, contains("water")))
  
  # group_by(cluster) %>% 
  # summarise(across(everything(), list(mean)))

# animals <- master %>% select(c(cluster, ta_code, chicken_count, cattle_count, goat_count, pig_count, sheep_count, livestock)) %>% 
#     group_by(cluster) %>% 
#     summarise(across(everything(), list(sum)))

## Stools
# stool <- master %>% 
#   group_by(cluster) %>% 
#   summarise(mean = mean(stool, na.rm=T),
#             sd = sd(pov_score, na.rm=T))
# stoolChi <- stool %>% filter(cluster=="Chikwawa")
# stoolMon <- stool %>% filter(cluster=="St Montfort")

sum_stool_3m <- master %>% 
  filter(!is.na(AGEGROUP_3mth) & AGEGROUP_3mth!="") %>% ## remove longitudinal for now as need to calculate age separately
  mutate(AGEGROUP_3mth = fct_relevel(AGEGROUP_3mth, c("[0,3]", "(3,6]","(6,9]","(9,12]","(12,15]", "(15,18]", "(18,21]","(21,24]",
                                               "(24,27]", "(27,30]","(30,33]","(33,36]","(36,39]", "(39,42]", "(42,45]","(45,48]",
                                               "(48,51]", "(51,54]","(54,57]","(57,60]"))) %>% 
  group_by(AGEGROUP_3mth) %>% 
  summarise(age_mean_stool = mean(calc_age_months, na.rm = T),
            age_sd_stool = sd(calc_age_months, na.rm=T),
            sum_pcr_pos = sum(PCR_pos_num, na.rm = T),
            sum_pcr_tested = sum(PCR_tested_bin, na.rm = T),
            PCR_pos_percent = (sum_pcr_pos/sum_pcr_tested)*100,
            sum_sal_pos = sum(Salmonella_num, na.rm = T),
            sum_stool_tested = sum(stool_tested, na.rm = T),
            Salm_pos_percent = sum_sal_pos/sum_stool_tested*100) %>% 
  mutate(across(where(is.numeric), round, 1))

sum_stool_all <- master %>% 
  summarise(age_mean_stool = mean(calc_age_months, na.rm = T),
            age_sd_stool = sd(calc_age_months, na.rm=T),
            sum_pcr_pos = sum(PCR_pos_num, na.rm = T),
            sum_pcr_tested = sum(PCR_tested_bin, na.rm = T),
            PCR_pos_percent = (sum_pcr_pos/sum_pcr_tested)*100,
            sum_sal_pos = sum(Salmonella_num, na.rm = T),
            sum_stool_tested = sum(stool_tested, na.rm = T),
            Salm_pos_percent = sum_sal_pos/sum_stool_tested*100) %>% 
  mutate(across(where(is.numeric), round, 1))


#age03m <- sum_stool_3m %>% filter(AGEGROUP_3mth=="(0-3]")
max_age_PCR <- sum_stool_3m %>% slice_max(PCR_pos_percent)
max_age_stool <- sum_stool_3m %>% slice_max(Salm_pos_percent)
min_age_PCR <- sum_stool_3m %>% slice_min(PCR_pos_percent)
min_age_stool <- sum_stool_3m %>% slice_min(Salm_pos_percent)

seq_sum <- seqdat %>% 
  group_by(cgmlst_subspecies) %>% 
  mutate(total = n()) %>% 
  ungroup() %>% 
  group_by(cgmlst_subspecies, `O antigen prediction`) %>%
  summarise(count = n(),
            `percent (%)` = round(count/total, digits=3)*100) %>% 
  slice_head(n=1) %>% 
  ungroup()

seq_sum_ent <- seq_sum %>% subset(cgmlst_subspecies=="enterica", c(`O antigen prediction`, count, `percent (%)`))
seq_sum_sal <- seq_sum %>% subset(cgmlst_subspecies=="salamae", c(`O antigen prediction`, count, `percent (%)`))

## MOnthly
seasonStool <- master %>% 
  mutate(month_yr = zoo::as.yearmon(d1s_data_date),
         month_yrfull = format(d1s_data_date, "%B %Y")) %>% 
  filter(!is.na(d1s_data_date)) %>% 
  group_by(month_yr, month_yrfull) %>% 
  summarise(age_mean_stool = mean(calc_age_months, na.rm = T),
            age_sd_stool = sd(calc_age_months, na.rm=T),
            sum_pcr_pos = sum(PCR_pos_num, na.rm = T),
            sum_pcr_tested = sum(PCR_tested_bin, na.rm = T),
            PCR_pos_percent = (sum_pcr_pos/sum_pcr_tested)*100,
            sum_sal_pos = sum(Salmonella_num, na.rm = T),
            sum_stool_tested = sum(stool_tested, na.rm = T),
            Salm_pos_percent = sum_sal_pos/sum_stool_tested*100) %>% 
  mutate(across(where(is.numeric), round, 1)) %>% 
  ungroup()

max_mth_PCR <- seasonStool %>% slice_max(PCR_pos_percent)
max_mth_stool <- seasonStool %>% slice_max(Salm_pos_percent)
min_mth_PCR <- seasonStool %>% slice_min(PCR_pos_percent)
min_mth_stool <- seasonStool %>% slice_min(Salm_pos_percent)

```

## Results

### SAiNTS cohort demographic and socioeconomic characteristics

`{r} print(n$n)` children aged 0-5 were recruited across `{r} print(sum(EA_clus$n))` EAs; `{r} print(chiN$n)` EA's in Chikwawa with a mean number of children per EA of `{r} print(round(chiN$mean_no, digits=1))`, range `{r} print(chiN$range)` equalling a total of `{r} print(nchik$n)` children. In St Montfort `{r} print(StMontN$n)` EA's with a mean number of children per EA of `{r} print(round(StMontN$mean_no, digits=1))`, range `{r} print(StMontN$range)`, with a total of `{r} print(nmont$n)` children. The geographical locations of the selected EA's are shown in @fig-selectedEA. The mean age of recruited children in Chikwawa was `{r} print(ageChi$age_mean)` (SD: `{r} print(ageChi$age_sd)`) and `{r} print(ageMon$age_mean)` (SD: `{r} print(ageMon$age_sd)`. The distribution of ages by 1 year age groups is shown in @tbl-cohortEA. Mean poverty scores were `{r} print(povChi$mean)` (SD: `{r} print(povChi$sd)`) in Chikwawa and `{r} print(povMon$mean)` (SD: `{r} print(povMon$sd)`) in St Montfort, and distribution in each cluster by poverty quintile is shown in @tbl-cohortEA .

::: {layout-nrow="2"}
![Selected Enumerations from the EPIMAL study census](figures/5_images/clusterMap.png){width="300"} ![Selected Enumeration Areas (EA) for the SAiNTS study from the EPIMAL Study Clusters (Chikwawa = Pink, St Montfort = Purple)](figures/5_images/selectedEA.png){#fig-selectedEA width="300"}
:::

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 6, fig.height=7, out.width = "6.5in"}
#| tbl-cap: Descriptive summary of children recruited across enumeration areas, by cluster. Chikwawa cluster did not receieve RTS,S/AS01 malaria vaccine. St Montfort did recieve RTS,S AS01 malaria vaccine
#| label: tbl-cohortEA

master %>% 
  select(c(cluster, calc_age_months, AGEGROUP_yr, gender, pov_score, pov_score_cat,toilet_type, washing_water_src, drinking_water_src, treat_dwater, mrdt_result, d90_stl_mrdt_result, malaria_film_result_d1, malaria_film_grade_d1, malaria_film_result_d90, malaria_film_grade_d90, hemocue_result, wfh_zscore, muac, sicklecell_test_result, hiv_exposed, hiv_test, hivtest_result, PCR_pos, Salmonella_bin, salm_fact)) %>% 
  mutate(sicklecell_test_result = ifelse(sicklecell_test_result=="", NA,sicklecell_test_result),
         hiv_exposed = ifelse(hiv_exposed=="", NA,hiv_exposed),
         hiv_test = ifelse(hiv_test=="", NA,hiv_test),
         hivtest_result = ifelse(hivtest_result=="", NA,hivtest_result),
         gender = ifelse(gender=="", NA,gender),
         pov_score_cat = ifelse(pov_score_cat=="", NA,pov_score_cat),
         mrdt_result = ifelse(mrdt_result=="", NA,mrdt_result),
         d90_stl_mrdt_result = ifelse(d90_stl_mrdt_result=="", NA,d90_stl_mrdt_result),
         malaria_film_result_d1 = ifelse(malaria_film_result_d1=="", NA,malaria_film_result_d1),
         malaria_film_result_d90 = ifelse(malaria_film_result_d90=="", NA,malaria_film_result_d90),
         PCR_pos = ifelse(PCR_pos=="", NA,PCR_pos)) %>%
  tbl_summary(by=cluster,
              statistic = list(all_continuous() ~ "{mean} ({sd})"),        # stats and format for continuous columns
                     # all_categorical() ~ "{n} / {N} ({p}%)"),   # stats and format for categorical columns
              digits = all_continuous() ~ 1,                              # rounding for continuous columns
              type   = all_categorical() ~ "categorical",                 # force all categorical levels to display
              label  = list(                                              # display labels for column names
                calc_age_months   ~ "Age (months)",                           
                AGEGROUP_yr ~ "Age (years)",
                gender    ~ "Gender",
                pov_score ~ "Poverty Score",
                pov_score_cat ~ "Poverty Score Quintile",
                mrdt_result  ~ "MRDT result (day 1)",
                d90_stl_mrdt_result  ~ "MRDT result (day 90)",
                malaria_film_result_d1  ~ "Malaria film result (day 1)",
                malaria_film_result_d90  ~ "Malaria film result (day 90)",
                malaria_film_grade_d1  ~ "Malaria film grade (day 1)",
                malaria_film_grade_d90  ~ "Malaria film grade (day 90)",
                hemocue_result  ~ "Haemoglobin",
                wfh_zscore    ~ "Weight-for-height z-score",
                muac    ~ "Mid-Upper Arm Circumference",
                sicklecell_test_result    ~ "Sickle Cell Disease",
                hiv_exposed~ "HIV exposed",
                hiv_test ~ "Had HIV test",
                hivtest_result ~ "HIV test result (if done)",
                PCR_pos ~ "Salmonella PCR result",
                Salmonella_bin ~ "Salmonella Culture result",
                salm_fact ~ "Salmonella sub-species (if culture positive"),
               missing_text = "Missing"                                    # how missing values should display
            ) %>% 
  bold_labels() %>%
  # convert to kableExtra
  as_kable_extra(booktabs = TRUE, longtable = TRUE) %>%
  # reduce font size to make table fit. 
  # you may also use the `latex_options = "scale_down"` argument here.
  kableExtra::kable_styling(latex_options = "scale_down", position='float_left') 
```

\newpage

<!-- changing the orientation to landscape --------------------------------- -->

```{=tex}
\KOMAoptions{usegeometry, paper=landscape,pagesize}
\recalctypearea
```
```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
#| tbl-cap: Host risk factor and NTS exposure distribution across recruited enumeration areas

sum_tab <- master %>% 
  group_by(cluster,ta_code) %>% 
  summarise(N = n(),
            MRDT_pos_d1 = sum(mrdt_num,na.rm=T),
            perMRDT_pos_d1 = round((MRDT_pos_d1/N)*100,digits=1),
            MRDT_pos_d90 = sum(d90_stl_mrdt_num,na.rm=T),
            perMRDT_pos_d90 = round((MRDT_pos_d90/N)*100,digits=1),
            sum_pcr_pos = sum(PCR_pos_num,na.rm = T),
            sum_pcr_tested = sum(PCR_tested_bin,na.rm = T),
            PCR_pos_percent = round((sum_pcr_pos/sum_pcr_tested)*100,digits=1),
            sum_sal_pos = sum(Salmonella_num, na.rm = T),
            sum_stool_tested = sum(stool_tested, na.rm = T),
            Salm_pos_percent = round((sum_sal_pos/sum_stool_tested)*100,digits=1)) %>% 
  select(-c(sum_pcr_tested, sum_stool_tested)) %>% 
  ungroup()  

# sum_tab2 <- tabulator(x = sum_tab, rows = c("cluster", "ta_code"), columns = c("N","MRDT_pos_d1","perMRDT_pos_d1","MRDT_pos_d90", "perMRDT_pos_d90","sum_pcr_pos","PCR_pos_percent","sum_sal_pos","Salm_pos_percent"))

sum_tab %>% 
  flextable() %>% 
  fontsize(size=8, part = "body") %>% 
  fontsize(size=8, part = "header") %>% 
  bold(i = 1, bold = TRUE, part = "header") %>%           
  # add_header_row(values = c("EA","N", "MRDT+ day 1 (n)", "MRDT+ day 1 (%)", "MRDT+ day 90 (n)", "MRDT+ day 90 (%)", "Salmonella PCR+ (n)","Salmonella PCR+ (%)","Salmonella Culture+ (n)","Salmonella Culture+ (%)",),
  #                top = TRUE) %>% 
  set_header_labels(
    cluster = "Cluster",
    ta_code = "EA",
    N = "N", 
    MRDT_pos_d1 = "MRDT+ day 1 (n)", 
    perMRDT_pos_d1 = "MRDT+ day 1 (%)",
    MRDT_pos_d90 = "MRDT+ day 90 (n)",
    perMRDT_pos_d90 = "MRDT+ day 90 (%)",
    sum_pcr_pos = "Salmonella PCR+ (n)",
    PCR_pos_percent = "Salmonella PCR+ (%)",
    sum_sal_pos = "Salmonella Culture+ (n)",
    Salm_pos_percent = "Salmonella Culture+ (%)"
    ) #%>% 
   # gdtools::register_gfont(family="Roboto")
  # merge_v(j = ~ cluster) %>% 
  # align(align = "center", part = "body")

```

\newpage

::: {layout-nrow="2"}
![MRDT positivity (%) by EA (dry season; May-Oct)](figures/5_images/MRDTdry.png){width="200"}

![MRDT positivity (%) by EA (wet season; Nov-Apr)](figures/5_images/MRDTwet.png){width="200"}

![Salmonella PCR positivity (%) (dry season; May-Oct)](figures/5_images/PCRdry.png){width="200"}

![Salmonella PCR positivity (%) (wet season; Nov - Apr)](figures/5_images/PCRwet.png){width="200"}
:::

\newpage

<!-- % changing the orientation to portrait again -------------------------- -->

```{=tex}
\KOMAoptions{usegeometry, paper=portrait,pagesize}
\recalctypearea
```
### Epidemiology of enteric non-typhoidal salmonella

*Salmonella* was detected in stools on microbiological culture in `{r} sum_stool_all$sum_sal_pos` of `{r} sum_stool_all$sum_stool_tested` (`{r} sum_stool_all$Salm_pos_percent` %) in healthy children at an average age of `{r} sum_stool_all$age_mean_stool` (SD: `{r} sum_stool_all$age_sd_stool`) months. Using pan-salmonella (TTR primer) PCR was positive in `{r} sum_stool_all$sum_pcr_pos` of `{r} sum_stool_all$sum_pcr_tested` (`{r} sum_stool_all$PCR_pos_percent` %) stools tested. The proportion of children with *salmonella* positive stools increased with age, ranging from `{r} min_age_PCR$Salm_pos_percent`% PCR positive at a mean age of `{r} min_age_PCR$age_mean_stool` months and `{r} min_age_stool$Salm_pos_percent`% culture positive at `{r} min_age_stool$age_mean_stool` months, to `{r} max_age_PCR$Salm_pos_percent`% PCR positive at a mean age of `{r} max_age_PCR$age_mean_stool` months and `{r} max_age_stool$Salm_pos_percent`% culture positive at `{r} max_age_stool$age_mean_stool` months (@tbl-stool3m_sum, @fig-stool3m). The percentage of stools that were salmonella culture and PCR positive by month of sample collection is shown in @fig-stoolseason . The months with the lowest percentages of children with salmonella culture positive stools was `{r} min_mth_stool$Salm_pos_percent`% in `{r} min_mth_stool$month_yrfull` and for PCR positive stools was `{r} min_mth_PCR$PCR_pos_percent`%, in `{r} min_mth_PCR$month_yrfull`. The months that had the highest salmonella culture positive stools was `{r} max_mth_stool$Salm_pos_percent`% in `{r} max_mth_stool$month_yrfull` and for PCR positive stools was `{r} max_mth_PCR$PCR_pos_percent`% in `{r} max_mth_PCR$month_yrfull`. Genome sequencing results were available for a subset of xx NTS isolates.

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=4.5, out.width = "6in"}
#| fig-cap: Percentage of children in each 3 month age group with A. Salmonella positive stool culture and B. Salmonella TTR PCR positive.
#| label: fig-stool3m

pPCR <- sum_stool_3m %>%
  ggplot() +
    geom_col(aes(x=AGEGROUP_3mth, y=PCR_pos_percent), fill="cornflowerblue") +
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="PCR positive (%)", x="Age (months)")
    #geom_text(stat='count', aes(label=..count..), vjust=-1) +
pcul <- sum_stool_3m %>%
  ggplot() +
    geom_col(aes(x=AGEGROUP_3mth, y=Salm_pos_percent), fill="orange") +
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Salmonella culture positive (%)", x="Age (months)")+
    scale_y_continuous(limits = c(0, 25))

pSerovar <- master %>% 
  filter(salm_fact!="Nil") %>% 
  mutate(AGEGROUP_3mth = fct_relevel(AGEGROUP_3mth, c("[0,3]", "(3,6]","(6,9]","(9,12]","(12,15]", "(15,18]", "(18,21]","(21,24]",
                                               "(24,27]", "(27,30]","(30,33]","(33,36]","(36,39]", "(39,42]", "(42,45]","(45,48]",
                                               "(48,51]", "(51,54]","(54,57]","(57,60]"))) %>% 
  mutate(`O antigen` = case_when(
    salm_fact=="Salmonella enteritidis" ~ "O9",
    salm_fact=="Salmonella typhimurium" ~ "O4",
    salm_fact=="Salmonella species" ~ "Other"
  )) %>% 
  # mutate(salm_fact = ifelse(salm_fact=="Nil", NA,salm_fact)) %>% 
  ggplot() +
  geom_bar(aes(fill=`O antigen`, x=AGEGROUP_3mth)) +
  scale_fill_manual(values = c("O9" = "orange", "O4" = "orchid", "Other" = "cyan3")) +  # Custom colors
  theme_bw() +
  theme(axis.text=element_text(size=6),
        axis.title=element_text(size=6,face="bold"),
        legend.text=element_text(size=6),
        legend.title=element_text(size=6),
        axis.text.x = element_text(angle=45, hjust=1)) +
  labs(y="Salmonella culture positive (n)", x="Age (months)") +
  scale_y_continuous(limits = c(0, 25))


# row1 <- cowplot::plot_grid(pcul, pPCR, labels="AUTO", label_size = 7)
# row2 <- cowplot::plot_grid(pSerovar, labels="C", label_size = 7)
# cowplot::plot_grid(row1, row2, nrow=2, ncol=1, rel_heights=c(0.9,1)) # 
cowplot::plot_grid(pSerovar, pPCR, nrow=2, labels="AUTO", label_size=7, rel_widths=c(1,0.85)) # 

```

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 4.5, fig.height=4, out.width = "5in"}
#| fig-cap: Percentage of children with positive salmonella stool by month of sample collection, A. Salmonella positive culture, B. Salmonella PCR (TTR primer) positive.
#| label: fig-stoolseason
#| 
pSalm_seas_n <- seasonStool %>% 
  ggplot() +
    geom_col(aes(x= factor(month_yr), y=Salm_pos_percent), fill="orchid")+
    # geom_bar(position="stack", stat='identity',  width = .7, colour="black", lwd=0.1) +
    # geom_text(aes(label = Percentage), colour="black", size = 2.5, position = position_stack(vjust = .5)) +
    labs(y="Culture positive (%)", x="Month") +
    scale_y_continuous(limits = c(0, 35))+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1))

pSalm_seas_p <- seasonStool %>% 
  ggplot() +
    geom_col(aes(x= factor(month_yr), y=PCR_pos_percent), fill="cornflowerblue")+
    # geom_bar(position="stack", stat='identity',  width = .7, colour="black", lwd=0.1) +
    # geom_text(aes(label = Percentage), colour="black", size = 2.5, position = position_stack(vjust = .5)) +
    labs(y="PCR positive (%)", x="Month") +
    theme_bw() +
    scale_y_continuous(limits = c(0, 35))+
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1))
cowplot::plot_grid(pSalm_seas_n, pSalm_seas_p, labels="AUTO", nrow=2, label_size = 7, rel_widths = c(1,0.8))
```

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
#| tbl-cap: Genome sequencing data for subset of non-typhoidal salmonella stool isolates from children recruited into the SAiNTS study
#| label: tbl-NTSgen

flextable(seq_sum_ent) %>%
  add_header_row(values=c("Salmonella enterica subspecies enterica (subspecies II)", "", "")) %>% 
  fontsize(size=8, part = "body") %>%
  fontsize(size=8, part = "header") %>%
  autofit()    

flextable(seq_sum_sal) %>%
  add_header_row(values=c("Salmonella enterica subspecies salamae (subspecies II)", "", "")) %>% 
  fontsize(size=8, part = "body") %>%
  fontsize(size=8, part = "header") %>%
  autofit()
```

### Epidemiology of host factor susceptibility

#### Malaria

Malaria RDT positivity at enrolment is 10.9% (262); 17.1% (210) in high and 4.3% (52) in low malaria transmission areas.

#### Anaemia

#### Malnutrition

Severe-acute malnutrition was present in 1.2% (31) by weight-for-height z-scores (\<-3 SD) and 3.2% (79) by mid-upper arm circumference \<12.5cm.

#### Sickle Cell

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
# 
# salm_micro <- micro_lims_uniq[grepl("S", micro_lims_uniq$organism), , drop = FALSE]
# 
# 
# ggplot(salm_micro, aes(x=month_yr_v1, fill=organism)) +
#   geom_bar() +
#   theme_bw() +
#   theme(axis.text=element_text(size=8),
#         axis.title=element_text(size=18,face="bold"),
#         axis.text.x = element_text(angle=45, hjust=1)) +
#   labs(y="Number of samples", x="Organism") +
#   geom_text(stat='count', aes(label=..count..), position = position_stack(0.5)) +
#   scale_y_continuous(limits = c(0, 20))


```

### Stool salmonella by month

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
# 
# ## Replace blank with NA
# unique_enrolled_v1_2_mal <- unique_enrolled_v1_2_mal %>%
#   mutate(across(c("organism.d1"), ~ifelse(.=="", NA, as.character(.))))
# 
# ### Percentage salmonella by month
# 
# unique_enrolled_v1_2_mal <- unique_enrolled_v1_2_mal %>%
#   mutate(
#     organism_cat = case_when(
#       str_detect(organism.d1, "Salmonella") ~ "Salmonella",
#       str_detect(organism.d1, "Typhi") ~ "Salmonella",
#       str_detect(organism.d1, "Citrobacter") ~ "Other",
#       str_detect(organism.d1, "Enterobacter") ~ "Other",
#       str_detect(organism.d1, "Proteus") ~ "Other",
#       str_detect(organism.d1, "Citrobacter yougae") ~ "Other",
#       str_detect(organism.d1, "C.Youngue") ~ "Other",
#       str_detect(organism.d1, "P.Mirabilis") ~ "Other",
#       str_detect(organism.d1, "R.orthnolityca") ~ "Other",
#       str_detect(organism.d1, "Proteus mirabilis") ~ "Other",
#       str_detect(organism.d1, " ") ~ "No growth",
#       is.na(organism.d1) ~ "No growth",
#       TRUE ~ organism.d1
#     )
#   )
# 
# 
# #table(unique_enrolled_v1_2_mal$organism_cat, unique_enrolled_v1_2_mal$month_fac)
# sal_per_mon <- data.frame(prop.table(table(unique_enrolled_v1_2_mal$organism_cat, unique_enrolled_v1_2_mal$month_yr_v1), 2))
# sal_per_mon <- sal_per_mon %>%
#   rename(
#     "Culture_result" = Var1,
#     "Month" = Var2,
#     "Percent" = Freq
#   ) %>%
#   mutate(
#     Percentage = round(Percent, digits = 2) *100
#   ) %>%
#   na.omit()
# 
# plotS_per <- ggplot(sal_per_mon, aes(x= factor(Month), y=Percentage, fill= Culture_result)) +
#               geom_bar(position="stack", stat='identity',  width = .7, colour="black", lwd=0.1) +
#               geom_text(aes(label = Percentage), colour="black", size = 2.5, position = position_stack(vjust = .5)) +
#               labs(y="% of stool samples processed", x="Month") + 
#               theme_bw() +
#               scale_fill_brewer("Culture Result", palette = "RdYlBu") +
#               theme(axis.text=element_text(size=8),
#                     axis.title=element_text(size=8,face="bold"),
#                     legend.text=element_text(size=8),
#                     legend.title=element_text(size=8),
#                     axis.text.x = element_text(angle=45, hjust=1))
# 
# ### Counts salmonella by month
# 
# stool_sum <- unique_enrolled_v1_2_mal %>% 
#   group_by(month_yr_v1, organism_cat) %>% 
#   summarise(count = sum(child_stool.d1)) %>%   # Within each Brand, sum all values in each Category
#   mutate( pos = cumsum(count)-0.5*count)
# 
# plotS_count <- ggplot(stool_sum, aes(x= factor(month_yr_v1), count, function(x)+sum(x), y=count, fill= factor(organism_cat))) +
#                 geom_bar(stat='identity',  width = .7, colour="black", lwd=0.1) +
#                 geom_text(aes(label=count), colour="white", size=2.5, position = position_stack(vjust = .5)) +
#                 labs(y="Number of enrolled participants", x="Month") + 
#                 theme_bw() +
#                 scale_fill_brewer("Culture Result", palette = "RdYlBu") +
#                 theme(axis.text=element_text(size=8),
#                       axis.title=element_text(size=8,face="bold"),
#                       legend.text=element_text(size=8),
#                       legend.title=element_text(size=8),
#                       axis.text.x = element_text(angle=45, hjust=1))
# 
# grid.arrange(plotS_count, plotS_per, nrow=2)
# 

```

### Salmonella by age

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
 # v1 <- subset(unique_enrolled_v1_2_mal)
# 
# age_salm_v1 <- data.frame(prop.table(table(v1$organism_cat, v1$AGEGROUP),2)) %>%
#   rename(Agegroup = Var2,
#          organism_cat = Var1,
#          Percentage = Freq) %>%
#   mutate(percentagelabelpos = ifelse(organism_cat=="Salmonella", Percentage/2, NA))
# 
# age_salm_v1$Percentage <- round(age_salm_v1$Percentage, digits = 3)
# 
# 
# ggplot(age_salm_v1, aes(x=Agegroup, y=Percentage, fill=factor(organism_cat))) +
#   geom_bar(position="fill", stat="identity", color='black', width=0.9) +
#   labs(y="Percentage", x="Age group (months)") + 
#   theme_bw() +
#   scale_y_continuous(labels = scales::percent) +
#   scale_fill_discrete(name = "Organism") +
#   geom_text(aes(label = paste0(Percentage*100,"%"), 
#                 y=percentagelabelpos), size = 3) +
#   ggtitle("Salmonella stool exposure by age") +
#   theme(axis.text=element_text(size=10),
#         axis.title=element_text(size=10,face="bold"),
#         legend.text=element_text(size=6),
#         legend.title=element_text(size=8),
#         axis.text.x = element_text(angle=45, hjust=1))
# 

```

### Epidemiology of risk factors; malaria, anaemia, malnutrition, sickle cell

\newpage

<!-- changing the orientation to landscape --------------------------------- -->

![](images/clipboard-157373631.png)

![](images/clipboard-2812805379.png)

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
#| fig-cap: MRDT positivity across recruited enumeration areas

# master$mrdt_result <- as.numeric(master$mrdt_result)
# #master$v1_month <- months(as.Date(master$v1_date))
# #master$v2_month <- months(as.Date(master$fup_crf_date))
# master <- master %>%
#   mutate(
#     month_yr_v1 = zoo::as.yearmon(v1_date) ,
#     month_yr_v2 = zoo::as.yearmon(fup_crf_date))
# 
# master$month_yr_v2 <- factor(master$month_yr_v2)
# master <- master %>%
#   mutate(month_yr_v2 = fct_expand(month_yr_v2, "Jan 2021", "Feb 2021","Mar 2021"),
#          month_yr_v2 = fct_relevel(month_yr_v2, c("Jan 2021", "Feb 2021","Mar 2021","Apr 2021", "May 2021", "Jun 2021","Jul 2021",
#                                                      "Aug 2021","Sep 2021","Oct 2021","Nov 2021","Dec 2021","Jan 2022")))
# 
# ### Malaria by month percentages
# 
# malaria_m <- subset(master, cluster_new=="Chikwawa")
# malaria_v1 <- data.frame(prop.table(table(malaria_m$mrdt_result, malaria_m$month_yr_v1),2))
# names(malaria_v1) <- c("mrdt", "month", "percentage")
# #malaria_v1$month_fac <- factor(malaria_v1$month, levels = month.name)
# # malaria_v1 <- malaria_v1 %>%
# #   arrange(month)
# malaria_v1$percentage <- round(malaria_v1$percentage, digits = 3)
# 
# malaria_v1$percentagelabelpos <- ifelse(malaria_v1$mrdt==1,
#                                         malaria_v1$percentage/2, 1 - malaria_v1$percentage/2)
# 
# plot1 <- ggplot(malaria_v1, aes(fill = factor(mrdt), x=month, y=percentage)) +
#   geom_bar(position="fill", stat="identity", color='black', width=0.9) +
#   labs(y="Percentage", x="Month") + 
#   theme_bw() +
#   scale_y_continuous(labels = scales::percent) +
#   scale_fill_discrete(name = "MRDT", labels=c("Neg", "Pos")) +
#   geom_text(aes(label = paste0(percentage*100,"%"), 
#                 y=percentagelabelpos), size = 1) +
#   ggtitle("Chikwawa Day 1 MRDT +") +
#   theme(axis.text=element_text(size=5),
#         axis.title=element_text(size=10,face="bold"),
#         legend.text=element_text(size=6),
#         legend.title=element_text(size=8),
#         axis.text.x = element_text(angle=45, hjust=1))
# 
# malaria_m <- subset(master, cluster_new=="St Montfort")
# malaria_v1 <- data.frame(prop.table(table(malaria_m$mrdt_result, malaria_m$month_yr_v1),2))
# names(malaria_v1) <- c("mrdt", "month", "percentage")
# #malaria_v1$month_fac = factor(malaria_v1$month, levels = month.name)
# # malaria_v1 <- malaria_v1 %>%
# #   arrange(month)
# malaria_v1$percentage <- round(malaria_v1$percentage, digits = 3)
# 
# malaria_v1$percentagelabelpos <- ifelse(malaria_v1$mrdt==1,
#                                         malaria_v1$percentage/2, 1 - malaria_v1$percentage/2)
# 
# plot2 <- ggplot(malaria_v1, aes(fill = factor(mrdt), x=month, y=percentage)) +
#   geom_bar(position="fill", stat="identity", color='black', width=0.9) +
#   labs(y="Percentage", x="Month") + 
#   theme_bw() +
#   scale_y_continuous(labels = scales::percent) +
#   scale_fill_discrete(name = "MRDT", labels=c("Neg", "Pos")) +
#   geom_text(aes(label = paste0(percentage*100,"%"), 
#                 y=percentagelabelpos), size = 1) +
#   ggtitle("St Montfort Day 1 MRDT +") +
#   theme(axis.text=element_text(size=5),
#         axis.title=element_text(size=10,face="bold"),
#         legend.text=element_text(size=6),
#         legend.title=element_text(size=8),
#         axis.text.x = element_text(angle=45, hjust=1))
# 
# 
# ##### Day 90 MRDT pos
# 
# malaria_m <- subset(master, cluster_new=="Chikwawa")
# malaria_v2 <- data.frame(prop.table(table(malaria_m$stl_mrdt_result, malaria_m$month_yr_v2),2))
# names(malaria_v2) <- c("mrdt", "month", "percentage")
# #malaria_v2$month_fac = factor(malaria_v2$month, levels = month.name)
# # malaria_v2 <- malaria_v2 %>%
# #    mutate(
# #     month_yr_v1 = format_ISO8601(v1_date, precision = "ym")) %>%
# #   arrange(month_yr_v1)
# malaria_v2$percentage <- round(malaria_v2$percentage, digits = 3)
# 
# malaria_v2$percentagelabelpos <- ifelse(malaria_v2$mrdt==1,
#                                         malaria_v2$percentage/2, 1 - malaria_v2$percentage/2)
# 
# plot3 <- ggplot(malaria_v2, aes(fill = factor(mrdt), x=month, y=percentage)) +
#   geom_bar(position="fill", stat="identity", color='black', width=0.9) +
#   labs(y="Percentage", x="Month") + 
#   theme_bw() +
#   scale_y_continuous(labels = scales::percent) +
#   scale_fill_discrete(name = "MRDT", labels=c("Neg", "Pos")) +
#   geom_text(aes(label = paste0(percentage*100,"%"), 
#                 y=percentagelabelpos), size = 1) +
#   ggtitle("Chikwawa Day 90 MRDT +") +
#   theme(axis.text=element_text(size=5),
#         axis.title=element_text(size=10,face="bold"),
#         legend.text=element_text(size=6),
#         legend.title=element_text(size=8),
#         axis.text.x = element_text(angle=45, hjust=1))
# 
# 
# 
# malaria_m <- subset(master, cluster_new=="St Montfort")
# malaria_v2 <- data.frame(prop.table(table(malaria_m$mrdt_result, malaria_m$month_yr_v2),2))
# names(malaria_v2) <- c("mrdt", "month", "percentage")
# malaria_v2$month_fac = factor(malaria_v2$month, levels = month.name)
# # malaria_v2 <- malaria_v2 %>%
# #   mutate(
# #     month_yr_v1 = format_ISO8601(v1_date, precision = "ym")) %>%
# #   arrange(month_yr_v1)
# malaria_v2$percentage <- round(malaria_v2$percentage, digits = 3)
# 
# malaria_v2$percentagelabelpos <- ifelse(malaria_v2$mrdt==1,
#                                         malaria_v2$percentage/2, 1 - malaria_v2$percentage/2)
# 
# plot4 <- ggplot(malaria_v2, aes(fill = factor(mrdt), x=month, y=percentage)) +
#   geom_bar(position="fill", stat="identity", color='black', width=0.9) +
#   labs(y="Percentage", x="Month") + 
#   theme_bw() +
#   scale_y_continuous(labels = scales::percent) +
#   scale_fill_discrete(name = "MRDT", labels=c("Neg", "Pos")) +
#   geom_text(aes(label = paste0(percentage*100,"%"), 
#                 y=percentagelabelpos), size = 1) +
#   ggtitle("St Montfort Day 90 MRDT +") +
#   theme(axis.text=element_text(size=5),
#         axis.title=element_text(size=10,face="bold"),
#         legend.text=element_text(size=6),
#         legend.title=element_text(size=8),
#         axis.text.x = element_text(angle=45, hjust=1))
# 
# grid.arrange(plot1, plot2, plot3, plot4, ncol=2, nrow=2)

```

![](images/clipboard-32344022.png)

Table high/ low malaria definitions

### Recruitment across VacciNTS sites

summary table of recruitment numbers

## Discussion

iNTS incidence increases rapidly in early life, but begins to fall between 18-36 months, despite continuing community exposure to *Salmonella* and malaria

Salmonella shedding [@gal-mor2018] NTS shedding rates......

Seasonality of eNTS and shows a seasonal pattern. Age-stratified distribution of salmonella stool culture and PCR positivity shows a sigmoidal distribution, showing low rates of exposure in the first 6 month of life (2% culture and 5% PCR positive), entering the exponential phase during the weaning period, and a plateau phase after 24 months (10% culture and 20% PCR positive).

For the subset of NTS stool isolates sequenced the only isolates that were O:4 or O:9 antigen positive (corresponding to the immunodominant O-antigens found on S. typhimurium and S. enteritidis respectively), were not Salmonella enterica subspecies enterica, but Salmonella enterica subspecies salamae (subspecies II) on genomic sequencing (@tbl-NTSgen), which have not been previously found to be associated with invasive NTS disease. Further testing on the remaining NTS isolates from stools in the SAINTS study is needed, but this change in epidemiology of NTS subspecies in enteric infections in children in Malawi, may also be correlated with the declines seen in invasive iNTS disease. Previous studies found S. typhimurium including ST313 strains in stool isolates from children 6-18 months in Malawi [@nyirenda2015], from data collected in 2012-14 - when iNTS disease incidence was higher (see chapter 3). Potentially the combination of reduced host susceptibility factors combined with changing epidemiology of NTS strains exposed to may explain the declines seen in iNTS disease. It is possible with the reduction in availability of host reservoirs in the immuno-compromised niche, that invasive NTS strains have become host adapted to, particularly in HIV infected adults where this niche is longer-term, compared to the transient immunosuceptibility associated with malaria, that the circulating invasive NTS strains have reduced, therefore further reducing iNTS incidence.

Malaria RTS,S/AS01 vaccine is highly confounded so not useful to look at in this setting. Mostly interested only in do kids have malaria exposure or not, regardless of cause. Not much impact of malaria

## Conclusion
