---
title: "Descriptive epidemiology of risk factors and enteric non-typhoidal salmonella in children under 5 years in Chikwawa"
format:    
  pdf:
    mainfont: Didot
    linestretch: 1.5   # Set line spacing to 1.5
    geometry:
      - top=20mm
      - bottom=20mm
      - left=20mm
      - right=20mm
    toc: true
    lof: true
    lot: true
    number-sections: true
    colorlinks: true
    include-in-header:
      text: |
        \usepackage{typearea}
    cite-method: citeproc
    bibliography: references.bib
    csl: diabetologia.csl
editor: visual
---

## Introduction

As illustrated in the previous chapter, understanding the relationship and interactions between non-typhoidal salmonella infections and host susceptibility factors such as malaria, HIV, severe acute malnutrition, and anaemia, is key to informing the focus of public health interventions. Previous work has been done to understand the underlying immune mechanisms leading to susceptibility by factors such as malaria [@nyirenda_immunological_2018, @nyirenda_loss_2017, @cunnington_suppression_2010, @gondwe_importance_2010], haemolysis [@cunnington2011], sickle cell disease [@hand_wl_deficiency_1977, @sanhadji1988], HIV [@gordon2002, @okoro2012, @gordon_invasive_2010, @maclennan_dysregulated_2010] and severe acute malnutrition (SAM) [@vásquez-garibay2002], as described in chapter 1. Asymptomatic carriage of NTS subclades associated with invasive disease have been demonstrated in 0.79% of stools from health children recruited as controls from a population living in Mukuru informal settlements in Kenya [@kariuki_high_2020]. NTS was found in the stools of 8-9% of children 6 - 18 months, 48% of which were *salmonella* typhimurium ST313, during monthly sampling in a longitudinal community cohort of 50 Malawian children [@nyirenda2015] as described in chapter 1. However, no large scale community stool testing for NTS in children under 5 years has been carried out in endemic settings, with simultaneous collection of data on host susceptibility factors such as malaria, malnutrition, anaemia and sickle cell disease, combined with markers of immune response to NTS. Previous seroepidemiological studies have relied on data collected at a single time point, and may miss crucial interactions of seasonal changes on exposure to enteric pathogens, seasonality of host susceptibility factors, and seasonality of immune responses to NTS enteric exposures.

This chapter sets out the descriptive epidemiological findings from the SAiNTS Malawi study, describing the spatial and temporal distribution of host susceptibility factors to NTS in children 0 - 5 years recruited in the Chikwawa district, Southern Malawi.

## Methods

As described in chapter 2, children were recruited prospectively randomly sampled from census data collected by the EPIMAL study across two clusters; St Montfort that was randomly allocated to receive the malaria RTS,S/AS01~E~ vaccine and Chikwawa that was randomly allocated to not receive the RTS,S/AS01~E~ vaccine. Two stage sampling was carried out using 2-stage sampling technique; Stage 1: sampling enumeration areas stratified by baseline malaria prevalence from the malaria indicator survey data for Chikwawa (@fig-MIS2017) [@malaria2017], and based on probability proportional to population size of the EA (PPS), and Stage 2: households containing children 0–5 years were randomly sampled from selected Enumeration Area (EA). Any child eligible in the household was invited to participate in the study. Children were recruited and samples were collected for stool, serology and host susceptibility data (malaria, anaemia, SAM, sickle cell) at day 1 and repeat serology and host susceptibility data at day 90. Serum samples were tested for lipopolysaccharide O-antigen immunoglobin-G antibody to the main serovars causing invasive NTS disease (*Salmonella* Typhimurium and Enteritidis). Host susceptibility factors were measured through anthropometric measurements, malaria (rapid diagnostic tests \[RDT\] and thick blood films), haemocue and sickle RDT. Stool samples were processed for Salmonella culture and pan-Salmonella PCR (TTR-primer). Data on medical history, vaccination history, socioeconomic status, water and sanitation and GPS location of households were collected by electronic case report forms on tablets at the household and blood sampling tent.

\newpage

<!-- changing the orientation to landscape --------------------------------- -->

```{=tex}
\KOMAoptions{usegeometry, paper=landscape,pagesize}
\recalctypearea
```
### Definitions

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "left", fig.width = 8, fig.height=7, out.width = "8.5in"}
#| tbl-cap: Integrated Management of Childhood Illness (IMCI) Guidelines 2019 (Organisa-tion 2014). *not in IMCI guidelines
#| label: tbl-IMCI
#| 
# setup
library(rio)
library(here)
library(tidyverse)
library(janitor)
library(Hmisc)
library(flextable)
library(gt)
library(gtsummary)
library(lubridate)

IMCI <- import(here("data", "5_data", "WatSan.xlsx"), which = "Sheet2") %>% 
  flextable() %>% 
  fontsize(size=8, part = "header") %>% 
  fontsize(size=8, part = "body") %>% 
  autofit()
IMCI
```

\newpage

<!-- changing the orientation to portrait --------------------------------- -->

```{=tex}
\KOMAoptions{usegeometry, paper=portrait,pagesize}
\recalctypearea
```
### Statistical methods

A proxy poverty score was derived as used in the SCALE trial [@feasey2023] to assess poverty in research studies in Malawi, with different variables selected to be appropriate for urban verses rural settings. Logistic regression was used to select the 10 most important variables from the Malawi 4^th^ Integrated Household Survey 2016-2017 (<https://microdata.worldbank.org/index.php/catalog/2936>) associated with poverty. Sequential models were run from 10 down to 6 variables to try and use the minimum variables possible to create a proxy poverty score. The model was run with and without the two ennumerator-defined variables (housing structure and floor composition) due to these variables being observed by fieldworkers and might not be able to be asked of participants. Bootstrapping was used to validate the model. The new model was compared with proxy poverty score variables from 1998 HIS, and the new variables improved the model fit. The coefficients from the final 10 variables were used as weights to form the final proxy poverty score (@tbl-povQ).

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 6, fig.height=7, out.width = "6.5in"}

master <- import(here("data", "5_data", "master.csv")) %>% 
  mutate(ta_code = substr(ta_code,5,8)) %>% 
  mutate( month_yr = zoo::as.yearmon(d1s_data_date),
          month_yrfull = format(d1s_data_date, "%B %Y"),
          month_yr_d90 = zoo::as.yearmon(d90_fup_crf_date),
          month_yrfull_d90 = format(d90_fup_crf_date, "%B %Y"),
          mrdt_num = ifelse(mrdt_result=="Positive",1,ifelse(mrdt_result=="Negative",0,NA)),
          mrdt_num_d90 = ifelse(d90_stl_mrdt_result=="Positive",1,ifelse(d90_stl_mrdt_result=="Negative",0,NA)),
          age_d90 = time_length(interval(dob, d90_fup_crf_date), "months"),
          age_d90 = ifelse(age_d90<70,age_d90,NA),
          AGEGROUP_3mth_d90 = cut(age_d90, breaks = seq(0,60,3)),
          AGEGROUP_3mth = factor(AGEGROUP_3mth, levels = c("[0,3]", "(3,6]","(6,9]","(9,12]","(12,15]", "(15,18]", "(18,21]","(21,24]","(24,27]", "(27,30]",
                                     "(30,33]","(33,36]","(36,39]", "(39,42]", "(42,45]","(45,48]", "(48,51]", "(51,54]","(54,57]","(57,60]"),
                          labels = c("0-3", "4-6", "7-9", "10-12","13-15", "16-18","19-21","22-24","25-27","28-30","31-33","34-36","37-39","40-42","43-45","46-48",
                                     "49-51","52-54","55-57","58-60")),
          AGEGROUP_3mth_d90 = factor(AGEGROUP_3mth_d90, levels = c("[0,3]", "(3,6]","(6,9]","(9,12]","(12,15]", "(15,18]", "(18,21]","(21,24]","(24,27]", "(27,30]",
                                     "(30,33]","(33,36]","(36,39]", "(39,42]", "(42,45]","(45,48]", "(48,51]", "(51,54]","(54,57]","(57,60]"),
                          labels = c("0-3", "4-6", "7-9", "10-12","13-15", "16-18","19-21","22-24","25-27","28-30","31-33","34-36","37-39","40-42","43-45","46-48",
                                     "49-51","52-54","55-57","58-60")),
          salm_fact2 = case_when(
            salm_fact=="Salmonella enteritidis" ~ "O9",
            salm_fact=="Salmonella typhimurium" ~ "O4",
            salm_fact=="Salmonella species" ~ "Salmonella other"),
          malnutrition_MUAC = case_when(
            muac<11.5 ~ "Severe acute malnutrition",
            muac>=11.5 & muac<=12.5 ~ "Moderate acute malnutrition",
            muac>12.5 ~ "No acute malnutrition"
          ),
          malnutrition_wfhz = case_when(
            wfh_zscore< -3 ~ "Severe acute malnutrition",
            wfh_zscore>= -3 & wfh_zscore <= -2 ~ "Moderate acute malnutrition",
            wfh_zscore> -2 ~ "No acute malnutrition"
            ),
          malnutrition_wfhz = factor(malnutrition_wfhz, levels=c("No acute malnutrition","Moderate acute malnutrition","Severe acute malnutrition")),
          malnutrition_MUAC = factor(malnutrition_MUAC, levels=c("No acute malnutrition","Moderate acute malnutrition","Severe acute malnutrition")),
          anaemia_cat = case_when(
            hemocue_result< 70 ~ "Severe anaemia",
            hemocue_result>= 70 & hemocue_result <= 110 ~ "Moderate anaemia",
            hemocue_result> 110 ~ "No anaemia"
            ),
          anaemia_cat_d90 = case_when(
            d90_v2_hemocue_result< 70 ~ "Severe anaemia",
            d90_v2_hemocue_result>= 70 & hemocue_result <= 110 ~ "Moderate anaemia",
            d90_v2_hemocue_result> 110 ~ "No anaemia"
          ),
           anaemia_cat = factor(anaemia_cat, levels=c("Severe anaemia","Moderate anaemia","No anaemia")),
           anaemia_cat_d90 = factor(anaemia_cat, levels=c("Severe anaemia","Moderate anaemia","No anaemia")),
          sickle_cell_res_comb = ifelse(sicklecell_test_result=="Positive"| d90_v2_sicklecell_result==3 | d90_v2_sicklecell_result==4, "Positive (HbSS/ HbSC)", 
                                        ifelse(sicklecell_test_result=="Negative"| d90_v2_sicklecell_result==1,"Negative",NA)),
          fever_meas = ifelse(temperature>=38,1,ifelse(temperature<38,0,NA)),
          fever_meas_d90 = ifelse(d90_v2_temperature>=38,1,ifelse(d90_v2_temperature<38,0,NA)),
          d90_v2_fever = ifelse(d90_v2_fever==2,0,d90_v2_fever)
  )
export(master, here("data", "5_data", "master3.csv"))

         # cluster = ifelse(cluster==1,"Chikwawa","St Montfort"))
seqdat <- import(here("data", "5_data", "2023.06.29.SAiNTS_seq_results.xlsx"))
watsan <- import(here("data", "5_data", "WatSan.xlsx"))
watsan_toishare <- watsan %>% 
  select(contains("share")) %>% 
  filter(if_all(everything(), ~ !is.na(.)))
watsan_toitype <- watsan %>% 
  select(contains("toilet_type")) %>% 
  filter(if_all(everything(), ~ !is.na(.))) %>% 
  mutate(toilet_type = as.numeric(toilet_type))
watsan_drink <- watsan %>% 
  select(contains("drinking")) %>% 
  filter(if_all(everything(), ~ !is.na(.)))# %>% 
 # mutate(drinking_water_src = as.numeric(drinking_water_src))
watsan_cook <- watsan %>% 
  select(contains("cooking")) %>% 
  filter(if_all(everything(), ~ !is.na(.)))
watsan_bathe <- watsan %>% 
  select(contains("bathing")) %>% 
  filter(if_all(everything(), ~ !is.na(.)))
watsan_treat1 <- watsan %>% 
  select(c(treat_dwater, treatW_desc)) %>% 
  filter(if_all(everything(), ~ !is.na(.)))
watsan_treat2 <- watsan %>% 
  select(c(treat_dwater_how, treat_dwater_how_desc)) %>% 
  filter(if_all(everything(), ~ !is.na(.)))

master2 <- master %>% 
  left_join(watsan_toishare) %>% 
  left_join(watsan_toitype) %>% 
  left_join(watsan_drink) %>% 
  left_join(watsan_cook) %>% 
  left_join(watsan_bathe) %>% 
  left_join(watsan_treat1) %>% 
  left_join(watsan_treat2)

export(master2, here("data", "5_data", "master2.csv"))

watsan_sum <- master2 %>%
  filter(toilet_type %in% c(2,3) | share_with %in% c(1,3) | drinking_water_src %in% c(9,10)) %>% 
  group_by(cluster) %>%
  summarise(
    toilet_type_desc2 = sum(toilet_type  %in% 2),
    toilet_type_desc3 = sum(toilet_type %in% 3),
    share_toilet_desc1 = sum(share_with  %in% 1),
    share_toilet_desc3 = sum(share_with %in% 3),
    drinking_desc9 = sum(drinking_water_src  %in% 9),
    drinking_desc10 = sum(drinking_water_src %in% 10),
    toilet_type_desc_per2 = (toilet_type_desc2 / n()) * 100,
    toilet_type_desc_per3 = (toilet_type_desc3 / n()) * 100,
    share_toilet_desc_per1 = (share_toilet_desc1 / n()) * 100,
    share_toilet_desc_per3 = (share_toilet_desc3 / n()) * 100,
    drinking_desc_per9 = (drinking_desc9 / n()) * 100,
    drinking_desc_per10 = (drinking_desc10 / n()) * 100) %>% 
  mutate(across(where(is.numeric), round, 1))
watsan_sumC <- watsan_sum %>% filter(cluster=="Chikwawa")
watsan_sumM <- watsan_sum %>% filter(cluster=="St Montfort")

# 
#             share_toilet_des,
#             drinking_desc)
watsancheck <- master2 %>%
  select(c(pid, share_with, share_toilet_des))

EA_clusEA <- master %>% 
  count(cluster, ta_code) 
EA_clus <- EA_clusEA %>% 
  count(cluster) 
n <- count(master)
nclus <- master %>% count(cluster)
nchik <- nclus %>% filter(cluster=="Chikwawa")
nmont <- nclus %>% filter(cluster=="St Montfort")

NclusEA <- EA_clusEA %>% 
  group_by(cluster) %>% 
  summarise(mean_no = mean(n, na.rm=T),
            range = range(n)) %>% 
  mutate(range_type = rep("low", "high",2)) %>% 
  pivot_wider(names_from = "range_type", values_from = "range") %>% 
  mutate(range = substr(low,2,10)) %>% 
  ungroup()

EAperClus <- EA_clus %>%  left_join(NclusEA)
chiN <- EAperClus %>% filter(cluster=="Chikwawa")
StMontN <- EAperClus %>% filter(cluster=="St Montfort")

ageclus <- master %>% 
  group_by(cluster) %>% 
  summarise(age_mean = mean(calc_age_months, na.rm=T),
            age_sd = sd(calc_age_months, na.rm=T)) %>% 
  mutate(across(where(is.numeric), round, 2))
ageChi <- ageclus %>% filter(cluster=="Chikwawa")
ageMon <- ageclus %>% filter(cluster=="St Montfort")

pov <- master %>% 
  group_by(cluster) %>% 
  summarise(mean = mean(pov_score, na.rm=T),
            sd = sd(pov_score, na.rm=T)) %>% 
  mutate(across(where(is.numeric), round, 2))
povChi <- pov %>% filter(cluster=="Chikwawa")
povMon <- pov %>% filter(cluster=="St Montfort")

# water <- master %>% select(c(cluster, toilet_type, contains("water")))
  
  # group_by(cluster) %>% 
  # summarise(across(everything(), list(mean)))

# animals <- master %>% select(c(cluster, ta_code, chicken_count, cattle_count, goat_count, pig_count, sheep_count, livestock)) %>% 
#     group_by(cluster) %>% 
#     summarise(across(everything(), list(sum)))

## Stools
# stool <- master %>% 
#   group_by(cluster) %>% 
#   summarise(mean = mean(stool, na.rm=T),
#             sd = sd(pov_score, na.rm=T))
# stoolChi <- stool %>% filter(cluster=="Chikwawa")
# stoolMon <- stool %>% filter(cluster=="St Montfort")


  
stoolsum_fun <- function(df){  
  df %>% 
  summarise(age_mean_stool = mean(calc_age_months, na.rm = T),
            age_sd_stool = sd(calc_age_months, na.rm=T),
            sum_pcr_pos = sum(PCR_pos_num, na.rm = T),
            sum_pcr_tested = sum(PCR_tested_bin, na.rm = T),
            PCR_pos_percent = (sum_pcr_pos/sum_pcr_tested)*100,
            sum_sal_pos = sum(Salmonella_num, na.rm = T),
            sum_stool_tested = sum(stool_tested, na.rm = T),
            Salm_pos_percent = sum_sal_pos/sum_stool_tested*100) %>% 
  mutate(across(where(is.numeric), round, 1))
}

sum_stool_3m <- master %>% 
  filter(!is.na(AGEGROUP_3mth) & AGEGROUP_3mth!="") %>% ## remove longitudinal for now as need to calculate age separately
  group_by(AGEGROUP_3mth) %>% 
  stoolsum_fun() %>% 
  ungroup()

sum_stool_3m_clus <- master %>% 
  filter(!is.na(AGEGROUP_3mth) & AGEGROUP_3mth!="") %>% ## remove longitudinal for now as need to calculate age separately
  group_by(cluster, AGEGROUP_3mth) %>% 
  stoolsum_fun() %>% 
  ungroup()


# sum_stool_3m_long_clst <- sum_stool_3m_clus %>% 
#   select(c(cluster, AGEGROUP_3mth, Salm_pos_percent, PCR_pos_percent)) %>% 
#   pivot_longer(cols = c(Salm_pos_percent, PCR_pos_percent), names_to = "Method", values_to = "Percent")
# sum_stool_3m_long <- sum_stool_3m %>% 
#   select(c(AGEGROUP_3mth, Salm_pos_percent, PCR_pos_percent)) %>% 
#   pivot_longer(cols = c(Salm_pos_percent, PCR_pos_percent), names_to = "Method", values_to = "Percent")

sum_stool_all <- master %>% 
  stoolsum_fun() %>% 
  ungroup()

#age03m <- sum_stool_3m %>% filter(AGEGROUP_3mth=="(0-3]")
max_age_PCR <- sum_stool_3m %>% slice_max(PCR_pos_percent)
max_age_stool <- sum_stool_3m %>% slice_max(Salm_pos_percent)
min_age_PCR <- sum_stool_3m %>% slice_min(PCR_pos_percent)
min_age_stool <- sum_stool_3m %>% slice_min(Salm_pos_percent)

seq_sum <- seqdat %>% 
  group_by(cgmlst_subspecies) %>% 
  mutate(total = n()) %>% 
  ungroup() %>% 
  group_by(cgmlst_subspecies, `O antigen prediction`) %>%
  summarise(count = n(),
            percent = round(count/total, digits=3)*100) %>% 
  slice_head(n=1) %>% 
  ungroup()

seq_sum_total <- seq_sum %>% summarise(count_tot = sum(count, na.rm=T))

seq_sum_ent <- seq_sum %>% subset(cgmlst_subspecies=="enterica", c(`O antigen prediction`, count, percent))
seq_sum_sal <- seq_sum %>% subset(cgmlst_subspecies=="salamae", c(`O antigen prediction`, count, percent))
tot_ent <- seq_sum_ent %>% summarise(count_tot = sum(count, na.rm=T))
tot_sal <-seq_sum_sal %>% summarise(count_tot = sum(count, na.rm=T))
O4_ent <- seq_sum_ent %>% filter(`O antigen prediction`=="4")
O4_sal <- seq_sum_sal %>% filter(`O antigen prediction`=="4")
O9_ent <- seq_sum_ent %>% filter(`O antigen prediction`=="9")
O9_sal <- seq_sum_sal %>% filter(`O antigen prediction`=="9")

## Monthly
seasonStool <- master %>% 
  filter(!is.na(d1s_data_date)) %>% 
  group_by(month_yr, month_yrfull) %>% 
  stoolsum_fun() %>% 
  ungroup()

sum_stoolseason_long <- seasonStool %>% 
  select(c(month_yr, month_yrfull, Salm_pos_percent, PCR_pos_percent)) %>% 
  pivot_longer(cols = c(Salm_pos_percent, PCR_pos_percent), names_to = "Method", values_to = "Percent")

max_mth_PCR <- seasonStool %>% slice_max(PCR_pos_percent)
max_mth_stool <- seasonStool %>% slice_max(Salm_pos_percent)
min_mth_PCR <- seasonStool %>% slice_min(PCR_pos_percent)
min_mth_stool <- seasonStool %>% slice_min(Salm_pos_percent)

seasonStool_clst <- master %>% 
  filter(!is.na(d1s_data_date)) %>% 
  group_by(cluster, month_yr, month_yrfull) %>% 
  stoolsum_fun() %>% 
  ungroup()

sum_stoolseason_long_clst <- seasonStool_clst %>% 
  select(c(cluster, month_yr, month_yrfull, Salm_pos_percent, PCR_pos_percent)) %>% 
  pivot_longer(cols = c(Salm_pos_percent, PCR_pos_percent), names_to = "Method", values_to = "Percent")

max_mth_PCR <- seasonStool %>% slice_max(PCR_pos_percent)
max_mth_stool <- seasonStool %>% slice_max(Salm_pos_percent)
min_mth_PCR <- seasonStool %>% slice_min(PCR_pos_percent)
min_mth_stool <- seasonStool %>% slice_min(Salm_pos_percent)

############ MRDT
## Age
mrdtsum_fun <- function(df){  
  df %>% summarise(age_mean = mean(calc_age_months, na.rm = T),
            age_sd = sd(calc_age_months, na.rm=T),
            sum_MRDT_posd1 = sum(mrdt_num, na.rm = T),
            sum_MRDT_testedd1 = sum(mrdt, na.rm = T),
            MRDT_pos_percentd1 = (sum_MRDT_posd1/sum_MRDT_testedd1)*100,
            sum_MRDT_posd90 = sum(mrdt_num_d90, na.rm = T),
            sum_MRDT_testedd90 = sum(d90_stl_mrdt, na.rm = T),
            MRDT_pos_percentd90 = (sum_MRDT_posd90/sum_MRDT_testedd90)*100,
            MDRT_tot = sum_MRDT_posd1+sum_MRDT_posd90,
            MRDT_all_per = MDRT_tot/ (sum_MRDT_testedd1+sum_MRDT_testedd90)*100) %>% 
  mutate(across(where(is.numeric), round, 1))
} 
sum_MRDT_3m_clus <- master %>% 
  filter(!is.na(AGEGROUP_3mth) & AGEGROUP_3mth!="") %>% ## remove longitudinal for now as need to calculate age separately
  group_by(cluster, AGEGROUP_3mth) %>% 
  mrdtsum_fun() %>% 
  ungroup()

sum_MRDT_3m <- master %>% 
  filter(!is.na(AGEGROUP_3mth) & AGEGROUP_3mth!="") %>% ## remove longitudinal for now as need to calculate age separately
  group_by(AGEGROUP_3mth) %>% 
  mrdtsum_fun() %>% 
  ungroup()

sum_MRDT_3m_c <- sum_MRDT_3m_clus %>% filter(cluster=="Chikwawa")
sum_MRDT_3m_m <- sum_MRDT_3m_clus %>% filter(cluster=="St Montfort")

min_age_MRDT_c <- sum_MRDT_3m_c %>% slice_min(MRDT_all_per)
max_age_MRDT_c <- sum_MRDT_3m_c %>% slice_max(MRDT_all_per)
min_age_MRDT_m <- sum_MRDT_3m_m %>% slice_min(MRDT_all_per, with_ties = F)
max_age_MRDT_m <- sum_MRDT_3m_m %>% slice_max(MRDT_all_per, with_ties = F)

seasonMRDT <- master %>% 
  filter(!is.na(d1s_data_date)) %>% 
  group_by(cluster, month_yr, month_yrfull) %>% 
  mrdtsum_fun() %>% 
  ungroup()

seasonMRDT_c <- seasonMRDT %>% filter(cluster=="Chikwawa")
seasonMRDT_m <- seasonMRDT %>% filter(cluster=="St Montfort")

min_mth_MRDT_c <- seasonMRDT_c %>% slice_min(MRDT_all_per)
max_mth_MRDT_c <- seasonMRDT_c %>% slice_max(MRDT_all_per)
min_mth_MRDT_m <- seasonMRDT_m %>% slice_min(MRDT_all_per, with_ties = F)
max_mth_MRDT_m <- seasonMRDT_m %>% slice_max(MRDT_all_per, with_ties = F)

MRDT <- master %>% 
  mrdtsum_fun() %>% 
  ungroup()

MRDT_clus <- master %>% 
  group_by(cluster) %>% 
  mrdtsum_fun() %>% 
  ungroup()

MRDT_clus_c <- MRDT_clus %>% filter(cluster=="Chikwawa")
MRDT_clus_m <- MRDT_clus %>% filter(cluster=="St Montfort")

### Anaemaia
other_clus <- master %>% 
  group_by(cluster) %>% 
  summarise(age_mean = mean(calc_age_months, na.rm = T),
            age_sd = sd(calc_age_months, na.rm=T),
            total = n(),
            hb_mean = mean(hemocue_result, na.rm = T),
            muac_mean = mean(muac, na.rm = T),
            wfh_zscore_mean = mean(wfh_zscore, na.rm = T),
            S.anaemia_cat_bin = ifelse(anaemia_cat=="Severe anaemia",1,ifelse(anaemia_cat=="Moderate anaemia"|anaemia_cat=="No anaemia",0,NA)),
            S.anaemia_cat_n = sum(S.anaemia_cat_bin, na.rm=T),
            S.anaemia_cat_per = (S.anaemia_cat_n/total)*100,
            M.anaemia_cat_bin = ifelse(anaemia_cat=="Moderate anaemia",1,ifelse(anaemia_cat=="Severe anaemia"|anaemia_cat=="No anaemia",0,NA)),
            M.anaemia_cat_n = sum(M.anaemia_cat_bin, na.rm=T),
            M.anaemia_cat_per = (M.anaemia_cat_n/total)*100,
            n.anaemia_cat_bin = ifelse(anaemia_cat=="No anaemia",1,ifelse(anaemia_cat=="Severe anaemia"|anaemia_cat=="Moderate anaemia",0,NA)),
            n.anaemia_cat_n = sum(n.anaemia_cat_bin, na.rm=T),
            n.anaemia_cat_per = (n.anaemia_cat_n/total)*100,
            hiv_exposed_bin = ifelse(hiv_exposed=="No",0,ifelse(hiv_exposed=="Yes",1,NA)),
            hiv_exposed_n = sum(hiv_exposed_bin, na.rm=T),
            hiv_exposed_per = (hiv_exposed_n/total)*100,
            hiv_test_bin = ifelse(hiv_test=="No",0,ifelse(hiv_test=="Yes",1,NA)),
            hiv_test_n = sum(hiv_test_bin, na.rm=T),
            hiv_test_per = (hiv_test_n/total)*100,
            hivtest_result_bin = ifelse(hivtest_result=="Negative",0,ifelse(hivtest_result=="Positive",1,NA)),
            hivtest_result_n = sum(hivtest_result_bin, na.rm=T),
            hivtest_result_per = (hivtest_result_n/total)*100,
            S.malnutrition_wfhz_bin = ifelse(malnutrition_wfhz=="Severe acute malnutrition",1,ifelse(malnutrition_wfhz=="Moderate acute malnutrition"| malnutrition_wfhz=="No acute malnutrition",0,NA)),
            S.malnutrition_wfhz_n = sum(S.malnutrition_wfhz_bin, na.rm=T),
            S.malnutrition_wfhz_per = (S.malnutrition_wfhz_n/total)*100,
            M.malnutrition_wfhz_bin = ifelse(malnutrition_wfhz=="Moderate acute malnutrition",1,ifelse(malnutrition_wfhz=="Severe acute malnutrition"| malnutrition_wfhz=="No acute malnutrition",0,NA)),
            M.malnutrition_wfhz_n = sum(M.malnutrition_wfhz_bin, na.rm=T),
            M.malnutrition_wfhz_per = (M.malnutrition_wfhz_n/total)*100,
            n.malnutrition_wfhz_bin = ifelse(malnutrition_wfhz=="No acute malnutrition",1,ifelse(malnutrition_wfhz=="Moderate acute malnutrition"| malnutrition_wfhz=="Severe acute malnutrition",0,NA)),
            n.malnutrition_wfhz_n = sum(n.malnutrition_wfhz_bin, na.rm=T),
            n.malnutrition_wfhz_per = (n.malnutrition_wfhz_n/total)*100,
            S.malnutrition_muac_bin = ifelse(malnutrition_MUAC=="Severe acute malnutrition",1,ifelse(malnutrition_MUAC=="Moderate acute malnutrition"| malnutrition_MUAC=="No acute malnutrition",0,NA)),
            S.malnutrition_muac_n = sum(S.malnutrition_muac_bin, na.rm=T),
            S.malnutrition_muac_per = (S.malnutrition_muac_n/total)*100,
            M.malnutrition_muac_bin = ifelse(malnutrition_MUAC=="Moderate acute malnutrition",1,ifelse(malnutrition_MUAC=="Severe acute malnutrition"| malnutrition_MUAC=="No acute malnutrition",0,NA)),
            M.malnutrition_muac_n = sum(M.malnutrition_muac_bin, na.rm=T),
            M.malnutrition_muac_per = (M.malnutrition_muac_n/total)*100,
            n.malnutrition_muac_bin = ifelse(malnutrition_MUAC=="No acute malnutrition",1,ifelse(malnutrition_MUAC=="Moderate acute malnutrition"| malnutrition_MUAC=="Severe acute malnutrition",0,NA)),
            n.malnutrition_muac_n = sum(n.malnutrition_muac_bin, na.rm=T),
            n.malnutrition_muac_per = (n.malnutrition_muac_n/total)*100,
            sicklecell_d1_bin = ifelse(sicklecell_test_result=="Positive",1,ifelse(sicklecell_test_result=="Negative",0,NA)),
            sicklecell_d1_n = sum(sicklecell_d1_bin, na.rm=T),
            sicklecelld1_ntest = sum(sicklecell_test, na.rm=T),
            # sicklecell_test_result_per = (sicklecell_test_result_n/total)*100,
            sicklecell_d90bin = ifelse(d90_v2_sicklecell_result==3,1,ifelse(d90_v2_sicklecell_result==1,0,NA)),
            sicklecell_d90_n = sum(sicklecell_d90bin, na.rm=T),
            sicklecell_d90test = ifelse(d90_v2_sicklecell_test==1,1,NA),
            sicklecelld90_ntest = sum(sicklecell_d90test, na.rm=T),
            sicklecell_n = sicklecell_d1_n+sicklecell_d90_n,
            sicklecell_test_per = sicklecell_n/(sicklecelld1_ntest+sicklecelld90_ntest)*100
            ) %>% 
  slice_head() %>% 
  mutate(across(where(is.numeric), round, 1)) %>% 
  ungroup()
other_clus_c <- other_clus %>% filter(cluster=="Chikwawa")
other_clus_m <- other_clus %>% filter(cluster=="St Montfort")

##################
## Malaria blood films
malaria_filmd1 <- master %>% 
  select(cluster, AGEGROUP_3mth, month_yr, month_yrfull,mrdt_result, mrdt_num, malaria_film_result_d1, malaria_film_grade_d1, fever, fever_meas) %>% 
  rename(malaria_film_result = malaria_film_result_d1,
         malaria_film_grade = malaria_film_grade_d1) %>% 
  mutate(day = "Day 1")

malaria_filmd90 <- master %>% 
  select(cluster, AGEGROUP_3mth_d90, month_yr_d90, month_yrfull_d90, mrdt_num_d90, d90_stl_mrdt_result, malaria_film_result_d90, malaria_film_grade_d90, d90_v2_fever, fever_meas_d90) %>% 
  rename(malaria_film_result = malaria_film_result_d90,
         malaria_film_grade = malaria_film_grade_d90,
         mrdt_result = d90_stl_mrdt_result,
         mrdt_num = mrdt_num_d90,
         fever = d90_v2_fever,
         fever_meas = fever_meas_d90,
         AGEGROUP_3mth = AGEGROUP_3mth_d90,
         month_yr = month_yr_d90,
         month_yrfull = month_yrfull_d90) %>% 
   mutate(day = "Day 90")

malaria_film_all <- bind_rows(malaria_filmd1, malaria_filmd90) %>% 
    mutate(malaria_film_result_num = ifelse(malaria_film_result=="Positive", 1, ifelse(malaria_film_result=="Negative",0,NA)))

malaria_film_done <- malaria_film_all %>% filter(malaria_film_result!="") %>% 
  mutate(mrdt_film_neg = ifelse(mrdt_num==0 & malaria_film_result_num==0,1,0),
         mrdt_pos_film_neg = ifelse(mrdt_num==1 & malaria_film_result_num==0,1,0),
         mrdt_neg_film_pos = ifelse(mrdt_num==0 & malaria_film_result_num==1,1,0),
         mrdt_film_pos = ifelse(mrdt_num==1 & malaria_film_result_num==1,1,0),
         mrdt_pos_fev = ifelse(mrdt_num==1 & fever==1,1,0),
         film_pos_fev = ifelse(malaria_film_result_num==1 & fever==1,1,0))

mrdtsum_fun <- function(df){  
  df %>%
  summarise(sum_film = sum(malaria_film_result_num, na.rm = T),
          sum_mrdt = sum(mrdt_num, na.rm = T),
          sum_mrdt_film_neg = sum(mrdt_film_neg, na.rm=T),
          sum_mrdt_film_pos = sum(mrdt_film_pos, na.rm=T),
          sum_mrdt_pos_film_neg = sum(mrdt_pos_film_neg, na.rm=T),
          sum_mrdt_neg_film_pos = sum(mrdt_neg_film_pos, na.rm=T),
          sum_fever = sum(fever, na.rm = T),
          sum_fever_meas = sum(fever_meas, na.rm=T),
          sum_mrdt_pos_fev = sum(mrdt_pos_fev, na.rm = T),
          sum_film_pos_fev = sum(film_pos_fev, na.rm = T),
          total = n(),
          film_per = (sum_film/total)*100,
          mrdt_per = (sum_mrdt/total)*100,
          mrdt_film_pos_per = (sum_mrdt_film_pos/total)*100,
          mrdt_film_neg_per = (sum_mrdt_film_neg/total)*100,
          mrdt_pos_film_neg_per = (sum_mrdt_pos_film_neg/total)*100,
          mrdt_neg_film_pos_per = (sum_mrdt_neg_film_pos/total)*100,
          fever_per = (sum_fever/total)*100,
          fever_meas_per = (sum_fever_meas/total)*100,
          mrdt_pos_fev_per = (sum_mrdt_pos_fev/sum_mrdt)*100,
          film_pos_fev_per = (sum_film_pos_fev/sum_film)*100
          ) %>% 
    mutate(across(where(is.numeric), round, 1))
}

malaria_film_age <- malaria_film_done %>% 
  group_by(AGEGROUP_3mth) %>% 
  mrdtsum_fun() %>% 
  ungroup()

malaria_film_age_clus <- malaria_film_done %>% 
  group_by(cluster, AGEGROUP_3mth) %>% 
  mrdtsum_fun() %>% 
  ungroup()

malaria_film_age_long_clst <- malaria_film_age_clus %>% 
  select(c(cluster, AGEGROUP_3mth, film_per, mrdt_per)) %>% 
  pivot_longer(cols = c(film_per, mrdt_per), names_to = "Method", values_to = "Percent")

malaria_film_seas <- malaria_film_done %>% 
  group_by(month_yr, month_yrfull) %>% 
  mrdtsum_fun() %>% 
  ungroup()

malaria_film_seas_clus <- malaria_film_done %>% 
  group_by(cluster, month_yr, month_yrfull) %>% 
  mrdtsum_fun() %>% 
  ungroup()

malaria_film_seas_long_clst <- malaria_film_seas_clus %>% 
  select(c(cluster, month_yr, month_yrfull, film_per, mrdt_per)) %>% 
  pivot_longer(cols = c(film_per, mrdt_per), names_to = "Method", values_to = "Percent")

#malaria_film_done_age %>% tbl_summary()
```

## Results

### SAiNTS cohort demographic and socioeconomic characteristics

`{r} print(n$n)` children aged 0-60 months were recruited across `{r} print(sum(EA_clus$n))` EAs; `{r} print(chiN$n)` EA's were recruited from the Chikwawa cluster with a mean number of children per EA of `{r} print(round(chiN$mean_no, digits=1))`, range: `{r} print(chiN$range)` equaling a total of `{r} print(nchik$n)` children. In the St Montfort cluster `{r} print(StMontN$n)` EA's with a mean number of children per EA of `{r} print(round(StMontN$mean_no, digits=1))`, range: `{r} print(StMontN$range)`, with a total of `{r} print(nmont$n)` children were recruited. The geographical locations of the selected EA's are shown in @fig-map & @fig-selectedEA . The mean age of recruited children in Chikwawa was `{r} print(ageChi$age_mean)` (SD: `{r} print(ageChi$age_sd)`) months and in St Montfort `{r} print(ageMon$age_mean)` (SD: `{r} print(ageMon$age_sd)`) months. The distribution of ages by 1 year age groups is shown in @tbl-cohortEA. Mean poverty scores were `{r} print(povChi$mean)` (SD: `{r} print(povChi$sd)`) in Chikwawa and `{r} print(povMon$mean)` (SD: `{r} print(povMon$sd)`) in St Montfort, and distribution in each cluster by poverty quintile is shown in @tbl-cohortEA. In Chikwawa `{r} watsan_sumC$toilet_type_desc3` (`{r} watsan_sumC$toilet_type_desc_per3`%) children lived in a household that used a pit latrine with slab, and `{r} watsan_sumC$toilet_type_desc2` (`{r} watsan_sumC$toilet_type_desc_per2`%) used pit latrine with wood/soil floor, compared to `{r} watsan_sumM$toilet_type_desc3` (`{r} watsan_sumM$toilet_type_desc_per3`%) and `{r} watsan_sumM$toilet_type_desc2` (`{r} watsan_sumM$toilet_type_desc_per2`%) respectively for children in St Montfort (@tbl-cohortEA). In Chikwawa `{r} watsan_sumC$share_toilet_desc1` (`{r} watsan_sumC$share_toilet_desc_per1`%) children lived in a household shared a toilet with more than three households, compared to `{r} watsan_sumC$share_toilet_desc3` (`{r} watsan_sumC$share_toilet_desc_per3`%) that did not share a toilet with another household; in St Montfort `{r} watsan_sumM$share_toilet_desc1` (`{r} watsan_sumM$share_toilet_desc_per1`%) shared a toilet with more than three households and `{r} watsan_sumM$share_toilet_desc3` (`{r} watsan_sumM$share_toilet_desc_per3`%) not sharing a toilet with another household respectively. Additional water, sanition and household animal variables are shown in @tbl-watsan .

![Location of SAiNTS study in Chikwawa, Southern Malawi](figures/5_images/clusterMap.png){#fig-map nrow="2" width="423"}

![Selected enumeration areas (EA); Chikwawa (pink), St Montfort (purple), selected EAs (dark purple)](figures/5_images/selectedEA.png){#fig-selectedEA width="421"}

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 6, fig.height=7, out.width = "6.5in"}
#| tbl-cap: Descriptive summary of children recruited across enumeration areas, by cluster. Chikwawa cluster did not receieve RTS,S/AS01 malaria vaccine. St Montfort did recieve RTS,S AS01 malaria vaccine
#| label: tbl-cohortEA

descript <- master2 %>% 
  select(c(cluster, calc_age_months, AGEGROUP_yr, gender, pov_score, pov_score_cat, toilet_type_desc, share_toilet_des, drinking_desc, keep_animals, rtss1, rtss2, rtss3, rtss4, mrdt_result, d90_stl_mrdt_result, malaria_film_result_d1, malaria_film_grade_d1, malaria_film_result_d90, malaria_film_grade_d90, anaemia_cat, malnutrition_MUAC, malnutrition_wfhz, sickle_cell_res_comb, hiv_exposed, hiv_test, hivtest_result, PCR_pos, Salmonella_bin, salm_fact2)) %>% 
  mutate(
        #sicklecell_test_result = ifelse(sicklecell_test_result=="", NA,sicklecell_test_result),
         hiv_exposed = ifelse(hiv_exposed=="", NA,hiv_exposed),
         hiv_test = ifelse(hiv_test=="", NA,hiv_test),
         hivtest_result = ifelse(hivtest_result=="", NA,hivtest_result),
         gender = ifelse(gender=="", NA,gender),
         pov_score_cat = ifelse(pov_score_cat=="", NA,pov_score_cat),
         mrdt_result = ifelse(mrdt_result=="", NA,mrdt_result),
         d90_stl_mrdt_result = ifelse(d90_stl_mrdt_result=="", NA,d90_stl_mrdt_result),
         malaria_film_result_d1 = ifelse(malaria_film_result_d1=="", NA,malaria_film_result_d1),
         malaria_film_result_d90 = ifelse(malaria_film_result_d90=="", NA,malaria_film_result_d90),
         PCR_pos = ifelse(PCR_pos=="", NA,PCR_pos)) %>%
  tbl_summary(by=cluster,
              statistic = list(all_continuous() ~ "{mean} ({sd})"),        # stats and format for continuous columns
                     # all_categorical() ~ "{n} / {N} ({p}%)"),   # stats and format for categorical columns
              digits = all_continuous() ~ 1,                              # rounding for continuous columns
              type   = all_categorical() ~ "categorical",                 # force all categorical levels to display
              label  = list(                                              # display labels for column names
                calc_age_months   ~ "Age (months)",                           
                AGEGROUP_yr ~ "Age (years)",
                gender    ~ "Gender",
                pov_score ~ "Poverty Score",
                pov_score_cat ~ "Poverty Score Quintile",
                toilet_type_desc   ~ "Toilet Type",                           
                share_toilet_des ~ "Shared toilet",
                drinking_desc    ~ "Drinking water",
                rtss1 ~ "RTS,S dose 1", 
                rtss2 ~ "RTS,S dose 2", 
                rtss3 ~ "RTS,S dose 3", 
                rtss4  ~ "RTS,S dose 4", 
                mrdt_result  ~ "MRDT result (day 1)",
                d90_stl_mrdt_result  ~ "MRDT result (day 90)",
                malaria_film_result_d1  ~ "Malaria film result (day 1)",
                malaria_film_result_d90  ~ "Malaria film result (day 90)",
                malaria_film_grade_d1  ~ "Malaria film grade (day 1)",
                malaria_film_grade_d90  ~ "Malaria film grade (day 90)",
                anaemia_cat  ~ "Anaemia",
                malnutrition_wfhz    ~ "Weight-for-height z-score",
                malnutrition_MUAC    ~ "Mid-upper arm circumference",
                sickle_cell_res_comb    ~ "Sickle Cell Disease",
                hiv_exposed~ "HIV exposed",
                hiv_test ~ "Had HIV test",
                hivtest_result ~ "HIV test result (if done)",
                PCR_pos ~ "Salmonella PCR result",
                Salmonella_bin ~ "Salmonella Culture result",
                salm_fact2 ~ "Salmonella O-antigen (if culture positive"),
               missing_text = "Missing"                                    # how missing values should display
            ) %>% 
  bold_labels() %>%
  # convert to kableExtra
  as_kable_extra(booktabs = TRUE, longtable = TRUE) %>%
  # reduce font size to make table fit. 
  # you may also use the `latex_options = "scale_down"` argument here.
  kableExtra::kable_styling(latex_options = "scale_down", position='float_left') 

descript

# # Convert to gt table to modify styling
# gt_tbl <- as_gt(descript)
# 
# # Modify the font style in the table
# gt_tbl <- gt_tbl %>%
#   tab_style(
#     style = list(
#       cell_text(font = "Times New Roman")   # Change font to 'Times New Roman'
#     ),
#     locations = cells_body()               # Apply to the body of the table
#   )
```

### Epidemiology of enteric non-typhoidal salmonella

*Salmonella* was detected in stools on microbiological culture in `{r} sum_stool_all$sum_sal_pos` of `{r} sum_stool_all$sum_stool_tested` (`{r} sum_stool_all$Salm_pos_percent` %) in healthy children at an average age of `{r} sum_stool_all$age_mean_stool` (SD: `{r} sum_stool_all$age_sd_stool`) months. Using pan-salmonella (TTR primer) PCR was positive in `{r} sum_stool_all$sum_pcr_pos` of `{r} sum_stool_all$sum_pcr_tested` (`{r} sum_stool_all$PCR_pos_percent` %) stools tested. The proportion of children with *salmonella* positive stools increased with age, ranging from `{r} min_age_PCR$Salm_pos_percent`% PCR positive in age group `{r} min_age_PCR$AGEGROUP_3mth` (mean: `{r} min_age_PCR$age_mean_stool`) months and `{r} min_age_stool$Salm_pos_percent`% culture positive in age group `{r} min_age_stool$AGEGROUP_3mth` (mean: `{r} min_age_stool$age_mean_stool`) months, to `{r} max_age_PCR$Salm_pos_percent`% PCR positive in age group `{r} max_age_PCR$AGEGROUP_3mth` (mean: `{r} max_age_PCR$age_mean_stool`) months and `{r} max_age_stool$Salm_pos_percent`% culture positive in age group `{r} max_age_stool$AGEGROUP_3mth` (mean: `{r} max_age_stool$age_mean_stool`) months (@tbl-stool3m_sum, @fig-stool3m). The percentage of stools that were salmonella culture and PCR positive by month of sample collection is shown in @fig-stoolseason. The months with the lowest percentages of children with salmonella culture positive stools was `{r} min_mth_stool$Salm_pos_percent`% in `{r} min_mth_stool$month_yrfull` and for PCR positive stools was `{r} min_mth_PCR$PCR_pos_percent`%, in `{r} min_mth_PCR$month_yrfull`. The months that had the highest salmonella culture positive stools was `{r} max_mth_stool$Salm_pos_percent`% in `{r} max_mth_stool$month_yrfull` and for PCR positive stools was `{r} max_mth_PCR$PCR_pos_percent`% in `{r} max_mth_PCR$month_yrfull`. Genome sequencing results were available for a subset of `{r} seq_sum_total$count_tot` NTS isolates, of which `{r} tot_ent$count_tot` were Salmonella enterica subspecies enterica (subspecies II) and `{r} tot_sal$count_tot` were Salmonella enterica subspecies salamae (subspecies II). Those isolates identified as Salmonella enterica subspecies salamae (subspecies II) were `{r} O9_sal$count` (`{r}  O9_ent$percent`%) identified with O9 antigen, and `{r} O4_sal$count` (`{r} O4_sal$percent`%) with O4 antigen. Those isolates identified as Salmonella enterica subspecies enterica (subspecies II) were `{r} O9_ent$count` (`{r} O9_ent$percent` %) identified with O9 antigen, and 0 with O4 antigen. Full list of serovars are shown in @tbl-seqdat_all.

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=7, out.width = "6in"}
#| fig-cap: Percentage of children in each 3 month age group with A. Salmonella positive stool culture, B. Salmonella TTR PCR positive and C. PCR positive by cluster.
#| label: fig-stool3m

pPCR <- sum_stool_3m %>%
  ggplot() +
    geom_col(aes(x=AGEGROUP_3mth, y=PCR_pos_percent), fill="cornflowerblue") +
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="PCR positive (%)", x="Age (months)")
    #geom_text(stat='count', aes(label=..count..), vjust=-1) +

pPCR_cl <- sum_stool_3m_clus %>%
  ggplot() +
    geom_col(aes(x=AGEGROUP_3mth, y=PCR_pos_percent), fill="orchid") +
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1),
          strip.text = element_text(size = 6)) +
    labs(y="PCR positive (%)", x="Age (months)") +
  facet_wrap(~ cluster)

pcul <- sum_stool_3m %>%
  ggplot() +
    geom_col(aes(x=AGEGROUP_3mth, y=Salm_pos_percent), fill="orange") +
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Salmonella culture positive (%)", x="Age (months)")+
    scale_y_continuous(limits = c(0, 25))

pSerovar <- master %>% 
  filter(salm_fact!="Nil") %>% 
  mutate(AGEGROUP_3mth = fct_relevel(AGEGROUP_3mth, c("[0,3]", "(3,6]","(6,9]","(9,12]","(12,15]", "(15,18]", "(18,21]","(21,24]",
                                               "(24,27]", "(27,30]","(30,33]","(33,36]","(36,39]", "(39,42]", "(42,45]","(45,48]",
                                               "(48,51]", "(51,54]","(54,57]","(57,60]"))) %>% 
  mutate(`O antigen` = case_when(
    salm_fact=="Salmonella enteritidis" ~ "O9",
    salm_fact=="Salmonella typhimurium" ~ "O4",
    salm_fact=="Salmonella species" ~ "Other"
  )) %>% 
  # mutate(salm_fact = ifelse(salm_fact=="Nil", NA,salm_fact)) %>% 
  ggplot() +
  geom_bar(aes(fill=`O antigen`, x=AGEGROUP_3mth)) +
  scale_fill_manual(values = c("O9" = "orange", "O4" = "orchid", "Other" = "cyan3")) +  # Custom colors
  theme_bw() +
  theme(axis.text=element_text(size=6),
        axis.title=element_text(size=6,face="bold"),
        legend.text=element_text(size=6),
        legend.title=element_text(size=6),
        axis.text.x = element_text(angle=45, hjust=1)) +
  labs(y="Salmonella culture positive (n)", x="Age (months)") +
  scale_y_continuous(limits = c(0, 25))

# pcomb <- sum_stool_3m_long %>% 
#   ggplot(aes(x=AGEGROUP_3mth, y=Percent, fill=Method)) +
#     geom_bar(position="dodge", stat="identity")+
#     scale_fill_manual(values = c("Salm_pos_percent" = "coral", "PCR_pos_percent" = "cornflowerblue"), labels=c("Culture Positive", "PCR Positive"))+
#     theme_bw() +
#     theme(axis.text=element_text(size=6),
#           axis.title=element_text(size=6,face="bold"),
#           axis.text.x = element_text(angle=45, hjust=1)) +
#     labs(y="Salmonella positive (%)", x="Age (months)")
    # scale_y_continuous(limits = c(0, 25))

# row1 <- cowplot::plot_grid(pcul, pPCR, labels="AUTO", label_size = 7)
# row2 <- cowplot::plot_grid(pSerovar, labels="C", label_size = 7)
# cowplot::plot_grid(row1, row2, nrow=2, ncol=1, rel_heights=c(0.9,1)) # 
cowplot::plot_grid(pSerovar, pPCR, pPCR_cl, nrow=3, labels="AUTO", label_size=7, rel_widths=c(1,0.85,0.85)) # 

```

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=6, out.width = "6in"}
#| fig-cap: Percentage of children with positive salmonella stool by month of sample collection, for Salmonella positive culture, and Salmonella PCR (TTR primer) positive, A. All children recruited into SAiNTS study, B. by study cluster.
#| label: fig-stoolseason
#| 
pSalm_seas_n <- seasonStool %>% 
  ggplot() +
    geom_col(aes(x= factor(month_yr), y=Salm_pos_percent), fill="orchid")+
    # geom_bar(position="stack", stat='identity',  width = .7, colour="black", lwd=0.1) +
    # geom_text(aes(label = Percentage), colour="black", size = 2.5, position = position_stack(vjust = .5)) +
    labs(y="Culture positive (%)", x="Month") +
    scale_y_continuous(limits = c(0, 35))+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1))

pSalm_seas_p <- seasonStool %>% 
  ggplot() +
    geom_col(aes(x= factor(month_yr), y=PCR_pos_percent), fill="cornflowerblue")+
    # geom_bar(position="stack", stat='identity',  width = .7, colour="black", lwd=0.1) +
    # geom_text(aes(label = Percentage), colour="black", size = 2.5, position = position_stack(vjust = .5)) +
    labs(y="PCR positive (%)", x="Month") +
    theme_bw() +
    scale_y_continuous(limits = c(0, 35))+
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          axis.text.x = element_text(angle=45, hjust=1))

pcomb <- sum_stoolseason_long %>% 
  ggplot(aes(x=factor(month_yr), y=Percent, fill=Method)) +
    geom_bar(position="dodge", stat="identity")+
    scale_fill_manual(values = c("Salm_pos_percent" = "orchid", "PCR_pos_percent" = "cornflowerblue"), labels=c("Culture Positive", "PCR Positive"))+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Salmonella positive (%)", x="Age (months)")

pcomb_clus <- sum_stoolseason_long_clst %>% 
  ggplot(aes(x=factor(month_yr), y=Percent, fill=Method)) +
    geom_bar(position="dodge", stat="identity")+
    scale_fill_manual(values = c("Salm_pos_percent" = "orange", "PCR_pos_percent" = "cyan3"), labels=c("Culture Positive", "PCR Positive"))+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Salmonella positive (%)", x="Age (months)") +
  facet_wrap(~cluster)

cowplot::plot_grid(pcomb, pcomb_clus,nrow=2, labels="AUTO", label_size = 7) # rel_widths = c(1,0.8)

#cowplot::plot_grid(pSalm_seas_n, pSalm_seas_p, pcomb,nrow=3, labels="AUTO", label_size = 7) # rel_widths = c(1,0.8)
```

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 7, fig.height=7, out.width = "7.5in"}
#| tbl-cap: Genome sequencing data for subset of non-typhoidal salmonella stool isolates from children recruited into the SAiNTS study
#| label: tbl-NTSgen

flextable(seq_sum_ent) %>%
  add_header_row(values=c("Salmonella enterica subspecies enterica (subspecies II)", "", "")) %>% 
  fontsize(size=8, part = "body") %>%
  fontsize(size=8, part = "header") %>%
  set_header_labels(
    count = "N",
    percent = "Percent (%)") %>% 
  autofit()    

flextable(seq_sum_sal) %>%
  add_header_row(values=c("Salmonella enterica subspecies salamae (subspecies II)", "", "")) %>% 
  fontsize(size=8, part = "body") %>%
  fontsize(size=8, part = "header") %>%
  set_header_labels(
    count = "N",
    percent = "Percent (%)") %>% 
  autofit()
```

### Epidemiology of host factor susceptibility

#### Malaria

Malaria rapid diagnostic test (MRDT) was positive in `{r} MRDT$MDRT_tot` (`{r} MRDT$MRDT_all_per`%) children over the whole duration of the study; `{r} MRDT_clus_c$MDRT_tot` (`{r} MRDT_clus_c$MRDT_all_per`%) in Chikwawa and `{r} MRDT_clus_m$MDRT_tot` (`{r} MRDT_clus_m$MRDT_all_per`%) in St Montfort. In Chikwawa, children in the age group `{r} min_age_MRDT_c$AGEGROUP_3mth` months (mean: `{r} min_age_MRDT_c$age_mean` months) had the lowest proportion of MRDT positive results `{r} min_age_MRDT_c$MDRT_tot` (`{r} min_age_MRDT_c$MRDT_all_per`%), and the age group `{r} max_age_MRDT_c$AGEGROUP_3mth` months (mean: `{r} max_age_MRDT_c$age_mean` months) had the highest `{r} max_age_MRDT_c$MDRT_tot` (`{r} max_age_MRDT_c$MRDT_all_per`%). In St Montfort, children in the age group `{r} min_age_MRDT_m$AGEGROUP_3mth` months (mean: `{r} min_age_MRDT_m$age_mean` months) had the lowest proportion of MRDT positive results `{r} min_age_MRDT_m$MDRT_tot` (`{r} min_age_MRDT_m$MRDT_all_per`%), and the age group `{r} max_age_MRDT_m$AGEGROUP_3mth` months (mean:`{r} max_age_MRDT_m$age_mean` months) had the highest `{r} max_age_MRDT_m$MDRT_tot` (`{r} max_age_MRDT_m$MRDT_all_per`%) (@fig-MRDT A). In Chikwawa, the month with the lowest proportion of MRDT positive results was `{r} min_mth_MRDT_c$month_yrfull` where there was `{r} min_mth_MRDT_c$MDRT_tot` (`{r} min_mth_MRDT_c$MRDT_all_per`%) children with positive MRDT results, compared to `{r} max_mth_MRDT_c$month_yrfull` with the highest percentage of `{r} max_mth_MRDT_c$MDRT_tot` (`{r} max_mth_MRDT_c$MRDT_all_per`%). In St Montfort, `{r} min_mth_MRDT_m$month_yrfull` had the lowest percentage children with positive MRDT results `{r} min_mth_MRDT_m$MDRT_tot` (`{r} min_mth_MRDT_m$MRDT_all_per`%) compared to `{r} max_mth_MRDT_m$month_yrfull` with the highest percentage `{r} max_mth_MRDT_m$MDRT_tot` (`{r} max_mth_MRDT_m$MRDT_all_per`%) (@fig-MRDT B). FILM results!!!

#### Anaemia

The mean haemoglobin was `{r} other_clus_c$hb_mean` g/L in Chikwawa and `{r} other_clus_m$hb_mean` g/L in St Montfort (@tbl-cohortEA). In Chikwawa there were `{r} other_clus_c$S.anaemia_cat_n` (`{r} other_clus_c$S.anaemia_cat_per`%) children with severe anaemia, `{r} other_clus_c$M.anaemia_cat_n` (`{r} other_clus_c$M.anaemia_cat_per`%) with moderate anaemia, and `{r} other_clus_c$n.anaemia_cat_n` (`{r} other_clus_c$n.anaemia_cat_per`%) with no anaemia; compared to `{r} other_clus_m$S.anaemia_cat_n` (`{r} other_clus_m$S.anaemia_cat_per`%) with severe, `{r} other_clus_m$M.anaemia_cat_n` (`{r} other_clus_m$M.anaemia_cat_per`%) with moderate and `{r} other_clus_m$n.anaemia_cat_n` (`{r} other_clus_m$n.anaemia_cat_per`%) with no anaemia in St Montfort (@tbl-cohortEA).

#### Malnutrition

The mean mid-upper arm circumference (MUAC) was `{r} other_clus_c$muac_mean`cm in Chikwawa and `{r} other_clus_m$muac_mean`cm in St Montfort, and mean weight-for-height z-score (WFHZ) was `{r} other_clus_c$wfh_zscore_mean` in Chikwawa and `{r} other_clus_m$wfh_zscore_mean` in St Montfort (@tbl-cohortEA). In Chikwawa, using WFHZ criteria (@tbl-IMCI) there were `{r} other_clus_c$S.malnutrition_wfhz_n` (`{r} other_clus_c$S.malnutrition_wfhz_per`%) children with severe acute malnutrition, `{r} other_clus_c$M.malnutrition_wfhz_n` (`{r} other_clus_c$M.malnutrition_wfhz_per`%) with moderate acute malnutrition, and `{r} other_clus_c$n.malnutrition_wfhz_n` (`{r} other_clus_c$n.malnutrition_wfhz_per`%) with no acute malnutrition; compared to `{r} other_clus_m$S.malnutrition_wfhz_n` (`{r} other_clus_m$S.malnutrition_wfhz_per`%) children with severe acute malnutrition, `{r} other_clus_m$M.malnutrition_wfhz_n` (`{r} other_clus_m$M.malnutrition_wfhz_per`%) with moderate acute malnutrition, and `{r} other_clus_m$n.malnutrition_wfhz_n` (`{r} other_clus_m$n.malnutrition_wfhz_per`%) with no acute malnutrition in St Montfort (@tbl-cohortEA). In Chikwawa, using MUAC criteria (@tbl-IMCI) there were `{r} other_clus_c$S.malnutrition_muac_n` (`{r} other_clus_c$S.malnutrition_muac_per`%) children with severe acute malnutrition, `{r} other_clus_c$M.malnutrition_muac_n` (`{r} other_clus_c$M.malnutrition_muac_per`%) with moderate acute malnutrition, and `{r} other_clus_c$n.malnutrition_muac_n` (`{r} other_clus_c$n.malnutrition_muac_per`%) with no acute malnutrition; compared to `{r} other_clus_m$S.malnutrition_muac_n` (`{r} other_clus_m$S.malnutrition_muac_per`%) children with severe acute malnutrition, `{r} other_clus_m$M.malnutrition_muac_n` (`{r} other_clus_m$M.malnutrition_muac_per`%) with moderate acute malnutrition, and `{r} other_clus_m$n.malnutrition_muac_n` (`{r} other_clus_m$n.malnutrition_muac_per`%) with no acute malnutrition in St Montfort (@tbl-cohortEA).

#### Sickle Cell

Rapid diagnostic test for haemoglobin S (in the absence of haemoglobin A) indicating sickle cell disease was positive in `{r} other_clus_c$sicklecell_n` (`{r} other_clus_c$sicklecell_test_per`%) in Chikwawa, and `{r} other_clus_m$sicklecell_n` (`{r} other_clus_m$sicklecell_test_per`%) in St Montfort (@tbl-cohortEA).

#### HIV

In Chikwawa `{r} other_clus_c$hiv_exposed_n` (`{r} other_clus_c$hiv_exposed_per`%) of children were HIV exposed, `{r} other_clus_c$hiv_test_n` (`{r} other_clus_c$hiv_test_per`%) had received an HIV test, and `{r} other_clus_c$hivtest_result_n` (`{r} other_clus_c$hivtest_result_per`%) of children who had been tested had a positive result. In St Montfort `{r} other_clus_m$hiv_exposed_n` (`{r} other_clus_m$hiv_exposed_per`%) of children were HIV exposed, `{r} other_clus_m$hiv_test_n` (`{r} other_clus_m$hiv_test_per`%) had received an HIV test, and `{r} other_clus_m$hivtest_result_n` (`{r} other_clus_m$hivtest_result_per`%) of children who had been tested had a positive result (@tbl-cohortEA).

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=5, out.width = "6in"}
#| fig-cap: Percentage of children with positive HHRP malaria rapid diagnostic tests (MRDT) A. by age group (months), and B. by month of sample collection
#| label: fig-MRDT

pMRDTs <- seasonMRDT %>% 
  ggplot(aes(x=factor(month_yr), y=MRDT_all_per)) + 
    geom_col(fill="cyan3")+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="MRDT positive (%)", x="Age (months)") +
  facet_wrap(~cluster)

pMRDTa <- sum_MRDT_3m_clus %>% 
  ggplot(aes(x=factor(AGEGROUP_3mth), y=MRDT_all_per)) + 
    geom_col(fill="orange")+
    theme_bw() +
    theme(axis.text=element_text(size=6),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="MRDT positive (%)", x="Month-Year") +
  facet_wrap(~cluster)

cowplot::plot_grid(pMRDTa, pMRDTs, nrow = 2, labels = "AUTO", label_size = 7)
```

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=5, out.width = "6in"}
#| fig-cap: Of children who recieved a malarial blood film (n=1331) at any visit, percentage of those with positive blood film compared with percentage with a positive malaria rapid diagnostic test (MRDT) A. by age group (months), and B. by month of sample collection
#| label: fig-film

pfilms <- malaria_film_seas_long_clst %>% 
  ggplot(aes(x=factor(month_yr), y=Percent, fill=Method)) + 
    geom_bar(position="dodge", stat="identity")+
    scale_fill_manual(values = c("film_per" = "orchid", "mrdt_per" = "cornflowerblue"), labels=c("Film Positive", "MRDT Positive"))+
    theme_bw() +
    theme(axis.text=element_text(size=5),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          strip.text = element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Malaria positive (%)", x="Month-Year") +
  facet_wrap(~cluster)

pfilma <- malaria_film_age_long_clst %>% 
  ggplot(aes(x=factor(AGEGROUP_3mth), y=Percent, fill=Method)) + 
    geom_bar(position="dodge", stat="identity")+
    scale_fill_manual(values = c("film_per" = "coral", "mrdt_per" = "seagreen"), labels=c("Film Positive", "MRDT Positive"))+
    theme_bw() +
    theme(axis.text=element_text(size=5),
          axis.title=element_text(size=6,face="bold"),
          legend.text=element_text(size=6),
          legend.title=element_text(size=6),
          strip.text = element_text(size=6),
          axis.text.x = element_text(angle=45, hjust=1)) +
    labs(y="Malaria positive (%)", x="Age (months)") +
  facet_wrap(~cluster)

cowplot::plot_grid(pfilma, pfilms, nrow = 2, labels = "AUTO", label_size = 7)
```

### Geospatial distribution of salmonella and malaria parasite prevalence

::: {#fig-geoMRDTSal layout-nrow="2"}
![MRDT positive results (%) by EA (dry season; May-Oct)](figures/5_images/MRDTdry.png){width="200"}

![MRDT positive (%) by EA (wet season; Nov-Apr)](figures/5_images/MRDTwet.png){width="200"}

![Salmonella PCR positive (%) (dry season; May-Oct)](figures/5_images/PCRdry.png){width="200"}

![Salmonella PCR positive (%) (wet season; Nov - Apr)](figures/5_images/PCRwet.png){width="200"}
:::

The distribution of salmonella PCR A summary of salmonella and MRDT results by individual EA is shown in @tbl-EAs & @fig-geoMRDTSal .

### Recruitment across VacciNTS sites

Recruitment across the four sub-Saharan African sites is shown in @tbl-VacciNTS .

```{r echo=FALSE, warning=FALSE, message=FALSE, fig.align = "centre", fig.width = 5.5, fig.height=7, out.width = "6in"}
#| tbl-cap: Recruitment across sub-Saharan African sites as part of the VacciNTS consortium seroepidemiology workpackage 
#| label: tbl-VacciNTS
VacciNTS <- import(here("data", "5_data", "VacciNTS_recruit.xlsx"))

VacciNTS %>% 
  flextable() %>% 
  fontsize(size=8, part="header") %>% 
  fontsize(size=8, part = "body") %>%
  autofit()
```

## Discussion

This is a large, potentially the largest, community serosurvey of children living in endemic iNTS settings combined with important meta-data on host susceptibility factors for invasive disease and potential risk factors for enteric infections. These results show an increasing enteric exposure to non-typhoidal *salmonella* (eNTS) with age, with children aged 22 - 60 months having higher prevalence of NTS detected in their stools, than children aged under 22 months. As demonstrated in chapter 3, iNTS incidence increases rapidly in early life to a peak between 9-12 months, but begins to decline thereafter with the steepest declines in incidence between 18-36 months. This pattern suggests that although children are continually being exposed to eNTS infections throughout early childhood, the prevalancec of eNTS infections alone does not explain iNTS disease incidence. This may be hypothesised to be due to a number of difference mechanisms. Firstly, eNTS infections may induce systemic immunity protective against iNTS disease following an initial, or several eNTS exposures; therefore older children will have been exposed to more eNTS infections and therefore will be more likely to have developed protective immunity against iNTS. However, it is possible that although eNTS infections may induce systemic immunity against iNTS disease, they may not induce local enteric mucosal immunity against non-invasive enteric infections, and therefore children may continue to have eNTS infections throughout childhood and adulthood, but have very low risk of developing iNTS disease. Secondly the prevalence of host susceptibility factors by age-group may be responsible for the patterns of age-stratified incidence observed in chapter 3. As discussed in chapter 4, the host susceptibility factors associated with highest population attributable fractions were malaria and severe anaemia and for children, and severe acute malnutrition, HIV and sickle cell disease to a lesser degree. However, the malaria parasite prevalence detected by malaria rapid diagnostic test in the SAiNTS study showed increasing malaria parasite prevalence with age, highest in children 24 to 60 months of age, therefore this alone does not explain the decreasing risk of iNTS disease in this age group. This higher malaria parasite prevalence in older children has been reported in other studies [@ujuju2023]. However, it has been shown that repeated malaria infections also induce malaria-specific immunity and older children in high malaria transmission areas are more likely to have asymptomatic plasmodium falciparum parasitaemia.

This would suggest that host susceptibility factors are a bigger determinant of the

Other seroprevalence studies undertaken in Kenya, using a multi-isotype ELISA assay (IgA, IgM, IgG) to be able to determine salmonella exposure from serum samples and used these to examine the relationship between NTS antibody responses and data on potential risk factors for enteric infections [@peter2023]. They reported an association between

There are few studies with large scale community stool sampling Salmonella shedding [@gal-mor2018] NTS shedding rates......

Seasonality of eNTS and shows a seasonal pattern. Age-stratified distribution of salmonella stool culture and PCR positive results shows a sigmoidal distribution, showing low rates of exposure in the first 6 month of life (2% culture and 5% PCR positive), entering the exponential phase during the weaning period, and a plateau phase after 24 months (10% culture and 20% PCR positive).

For the subset of NTS stool isolates sequenced the only isolates that were O:4 or O:9 antigen positive (corresponding to the immunodominant O-antigens found on S. typhimurium and S. enteritidis respectively), were not Salmonella enterica subspecies enterica, but Salmonella enterica subspecies salamae (subspecies II) on genomic sequencing (@tbl-NTSgen), which have not been previously found to be associated with invasive NTS disease. Further testing on the remaining NTS isolates from stools in the SAINTS study is needed, but this change in epidemiology of NTS subspecies in enteric infections in children in Malawi, may also be correlated with the declines seen in invasive iNTS disease. Previous studies found S. typhimurium including ST313 strains in stool isolates from children 6-18 months in Malawi [@nyirenda2015], from data collected in 2012-14 - when iNTS disease incidence was higher (see chapter 3). Potentially the combination of reduced host susceptibility factors combined with changing epidemiology of NTS strains exposed to may explain the declines seen in iNTS disease. It is possible with the reduction in availability of host reservoirs in the immuno-compromised niche, that invasive NTS strains have become host adapted to, particularly in HIV infected adults where this niche is longer-term, compared to the transient immunosuceptibility associated with malaria, that the circulating invasive NTS strains have reduced, therefore further reducing iNTS incidence.

### Limitations

Malaria RTS,S/AS01 vaccine is highly confounded so not useful to look at in this setting. Mostly interested only in do kids have malaria exposure or not, regardless of cause. Not much impact of malaria

## Conclusion
